---
title: ''
csl: msphere.csl
fontsize: 11pt
output:
  word_document: default
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
  html_document:
    df_print: paged
geometry: margin=1.0in
bibliography: references.bib
---


```{r knitr_settings, eval=TRUE, echo=FALSE, cache=FALSE, warning=FALSE, message=FALSE, tidy=TRUE}
options(tidyverse.quiet = TRUE)

library(tidyverse)
library(knitr)

opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("message" = FALSE)
opts_chunk$set("cache" = FALSE)

inline_hook <- function(x){
	print(x)

	if(is.list(x)){
		x <- unlist(x)
	}

	if(is.numeric(x)){
		if(abs(x - round(x)) < .Machine$double.eps^0.5){
			paste(format(x,big.mark=',', digits=0, scientific=FALSE))
		} else {
			paste(format(x,big.mark=',', digits=2, nsmall=2, scientific=FALSE))
		}
	} else {
    	paste(x)      
	}
}
knitr::knit_hooks$set(inline=inline_hook)
```



\vspace{35mm}

# Initial gut microbiota and response to antibiotic perturbation influence *Clostridioides difficile* colonization in mice

\vspace{35mm}


Sarah Tomkovich${^1}$, Joshua M.A. Stough${^1}$, Lucas Bishop${^1}$, Patrick D. Schloss^1$\dagger$^

\vspace{40mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI 48109




\newpage
\linenumbers


## Abstract
The microbiota plays a key role in determining susceptibility to *Clostridioides difficile* infections (CDIs). However, much of the mechanistic work examining CDIs in mouse models use a single university colony or vendor, which have lower inter-individual microbiota variation compared to humans. We treated mice from 6 different colony sources (2 University of Michigan colonies and 4 vendors) with a single clindamycin dose, followed by *C. difficile* challenge 1 day later and measured *C. difficile* colonization levels through 9 days post-infection. The microbiota was profiled throughout the experiment via 16S rRNA gene sequencing analysis to examine variation across colony sources and alterations due to clindamycin treatment and *C. difficile* challenge. While all sources of mice were colonized 1-day post-infection, variation in *C. difficile* colonization levels emerged from days 3-7 post-infection with 3 sources colonized with *C. difficile* for longer and at higher levels. We identified bacterial taxa with different relative abundances across colony sources throughout the experiment, as well as taxa that were consistently impacted by clindamycin treatment. We created bacterial community-based logistic regression models that successfully classified mice based on their day 7 *C. difficile* colonization status. Importantly, after examining the taxa that were most important to the classification models, we identified a subset of key taxa that varied across colony sources (*Bacteroides*, *Deferribacteraceae*), were altered by clindamycin (*Porphyromonadaceae*, *Ruminococcaceae*), or both (*Enterobacteriaceae*, *Enterococcus*, *Bifidobacteriaceae*, *Coriobacteriaceae*, *Lachnospiraceae*, and *Verrucomicrobiaceae*). These results suggest the response of the initial gut microbiota to clindamycin treatment influences *C. difficile* 630 colonization dynamics.

## Importance
*Clostridioides difficile* is a leading nosocomial infection. Although the microbiota has been established as a key risk factor, there is variation in who becomes asymptomatically colonized, develops an infection, or has an infection with adverse outcomes. *C. difficile* infection (CDI) mouse models are widely used to answer a variety of *C. difficile* pathogenesis questions. However, the inter-individual variation between mice is less than what is observed in humans, particularly if just one source of mice is used. In this study, we administered clindamycin to mice from 6 different colony sources and challenged them with *C. difficile*. Interestingly, only a subset of the taxa that vary across sources were associated with how long *C. difficile* was able to colonize. Future studies examining the interplay between the microbiota and *C. difficile* should consider using mice from multiple sources to narrow down the microbes driving the observed phenotypes and reflect human interindividual variation.

\newpage

## Introduction

Antibiotics are a clear risk factor for *Clostridioides difficile* infections (CDIs), but there is variation in who goes on to develop severe or recurrent CDIs after exposure [@Teng2019; @Kelly2012]. Additionally, asymptomatic colonization, where *C. difficile* is detectable, but symptoms are absent has been documented in infants and adults [@Zacharioudakis2015; @Crobach2018]. The intestinal microbiome has been implicated in asymptomatic colonization [@Zhang2015; @VanInsberghe2020], susceptibility to CDIs [@Mancabelli2017, @Duvallet2017], and adverse CDI outcomes [@Seekatz2016; @Khanna2016; @Pakpour2017; @Lee2020]. 

Mouse models of CDIs have been a great tool for understanding *C. difficile* pathogenesis [@Hutton2014]. The number of CDI mouse model studies has grown substantially since Chen et al. published their C57BL/6 model in 2008, which disrupted the gut microbiota with antibiotics to enable *C. difficile* colonization, diarrhea, and weight loss [@Chen2008]. CDI mouse models have been used to examine translationally relevant questions regarding *C. difficile*, including the role of the microbiota and efficacy of potential therapeutics for treating CDIs [@Best2012]. However, microbiome variation between lab mice is much less than the variation observed between humans [@Baxter2014; @Nagpal2018]. Additionally, studying the contribution of the microbiota to a particular disease phenotype in one set of lab mice after the same perturbation could yield a number of findings of which only a fraction may be driving the phenotype. 

In the past, our group has attempted to introduce more microbiome variation into the CDI mouse model by using a variety of antibiotic treatments [@Reeves2011; @Schubert2015; @Jenior2017; @Jenior2018]. An alternative approach to maximize microbiome variation is to use mice from multiple sources [@Velazquez2019; @Osbelt2020]. Microbiome differences between different mouse vendors have been well documented and shown to influence susceptibility to a variety of diseases [@Stough2016; @Alegre2019], including enteric infections [@EtienneMesmin2017; @Velazquez2019; @Osbelt2020; @Lai2020; @Thiemann2017; @Rolig2013; @Ge2018]. Additionally, different research groups have observed different CDI outcomes in mice despite using similar models and the microbiome has been proposed as one factor potentially mediating susceptibility [@Hutton2014; @Lawley2013; @Reeves2011; @Buffie2011; @Jenior2018; @Buffie2014]. Here we examined how variations in the baseline microbiome and responses to clindamycin treatment in C57BL/6 mice from six different sources influenced susceptibility to *C. difficile* colonization and the time needed to clear the infection.

```{r cfu}
cfu_stats_all <- read_tsv("../data/process/cfu_stats_all_days.tsv")
padj_d1_cfu <- cfu_stats_all %>% filter(day == 1) %>% pull(p.value.adj) 
mean_d1 <- cfu_stats_all %>% filter(day == 1) %>% select(-day, -statistic, -p.value, -parameter, -method, -p.value.adj) %>% pivot_longer(everything(), values_to = "mean_cfu", names_to = "Source") %>% arrange(mean_cfu)
min_mean_d1 <- mean_d1 %>%  select(mean_cfu) %>% min
max_mean_d1 <- mean_d1 %>% select(mean_cfu) %>% max

cfu_stats_sig <- cfu_stats_all %>% filter(p.value.adj < 0.05)
max_padj_d3to7 <- cfu_stats_sig %>% select(p.value.adj) %>% max
```
## Results

**Clindamycin treatment renders all mice susceptible to *C. difficile* 630 colonization regardless of colony source.** To test how the microbiotas of mice from different colony sources impact colonization dynamics after clindamycin exposure, we utilized C57BL/6 mice from 6 different sources: two colonies from the University of Michigan (the Young and Schloss lab colonies), the Jackson Laboratory, Charles River Laboratories, Taconic Biosciences, and Envigo (which was formerly Harlan). These 4 vendors were chosen because they represent commonly used vendors for CDI studies in mice [@Spinler2016; @Markey2018; @McKee2018; @Yamaguchi2020; @Stroke2018; @EtienneMesmin2017; @Quigley2019; @Mullish2019]. After a 13-day acclimation period for the mice ordered from vendors, all mice were treated with 10 mg/kg clindamycin via intraperitoneal injection and one day later challenged with 10^3^ *C. difficile* 630 spores (Fig. 1A). Clindamycin was chosen because we have previously demonstrated mice are rendered susceptible, but consistently cleared the CDI within 9 days [@Jenior2018; @Tomkovich2019], clindamycin is frequently implicated with human CDIs [@Guh2018], and is also part of the antibiotic treatment for the frequently cited 2008 CDI mouse model [@Chen2008]. The day after infection, *C. difficile* was detectable in all mice at a similar level (Mean CFU range: `r format(min_mean_d1, digits = 2, scientific=TRUE)`-`r format(max_mean_d1, scientific=TRUE, digits = 2)`; *P*~FDR~ = `r padj_d1_cfu`), indicating clindamycin rendered all mice susceptible regardless of colony source (Fig. 1B). Interestingly, variation in *C. difficile* CFU levels across sources of mice emerged from days 3-7 post-infection (all *P*~FDR~ $\le$ `r max_padj_d3to7`; Fig. 1B and Table S1), suggesting mouse colony source is associated with *C. difficile* clearance. We conducted two experiments approximately 3 months apart and combined the data because the colonization dynamics across sources of mice varied similarily in both experiments (Fig. S1). Although *C. difficile* 630 causes mild symptoms in mice comparied to other *C. difficile* strains [@Theriot2011], we also saw that weight change significantly varied across sources of mice with the most weight lost two days post-infection (Fig. 1C and Table S2). Interestingly, mice ordered from Jackson, Taconic, and Envigo tended to lose the most weight, have higher *C. difficile* CFU levels and take longer to clear the infection compared to the other sources of mice, which was particularly evident 7 days post-infection (Fig. 1C and D). Importantly, there was also one Jackson and one Envigo mouse that died between 1- and 3-days post-infection. Thus, clindamycin rendered all mice susceptible to *C. difficile* 630 colonization, regardless of colony source, but variation across sources emerged with 3 out of 6 sources taking longer to clear *C. difficile*.

```{r alpha}
richness_stats_all <- read_tsv("../data/process/richness_stats_all_days.tsv")
padj_dn1_richness <- richness_stats_all %>% filter(day == -1) %>% pull(p.value.adj)

shannon_stats_all <- read_tsv("../data/process/shannon_stats_all_days.tsv")
padj_dn1_shannon <- shannon_stats_all %>% filter(day == -1) %>% pull(p.value.adj)
```
**Bacterial communities vary across mouse colony sources.** Given the well known variation in mouse microbiomes across vendors and university colonies [@Alegre2019], we hypothesized that the variation in *C. difficile* clearance could be explained by microbiota variation across the 6 sources. We used 16S rRNA gene sequencing to characterize the fecal bacterial communities from the mice over the course of the experiment. Since antibiotics and other risk factors of CDIs are associated with decreased microbiota diversity [@Ross2016], we first examined alpha diversity measures across the 6 sources of mice. Examining the bacterial communities at baseline, prior to clindamycin treatment there was a significant difference in the number of observed OTUs (*P*~FDR~ = `r padj_dn1_richness`), but not Shannon diversity index (*P*~FDR~ = `r padj_dn1_shannon`) across sources of mice (Fig. 2A-B and Table S3). As expected, clindamycin treatment decreased richness and Shannon diversity across all sources of mice, and communities started to recover 1 day post-infection (Fig. 2C-D). Interestingly, significant differences in diversity metrics across sources (*P*~FDR~ < 0.05) emerged after both clindamycin and *C. difficile* infection, with Charles River mice having higher richness and Shannon Diversity than most of the other groups (Fig 2C-F and Table S4). While Charles River mice had more diverse microbiotas, Young and Schloss lab mice were also able to clear *C. difficile* faster, suggesting microbiota diversity alone does not explain the observed variation in *C. difficile* colonization across vendors.
```{r adonis}
adonis_stats_all <- read_tsv("../data/process/adonis_all.tsv")
r_sq_ordered_adonis <- adonis_stats_all %>% arrange(desc(r_sq))
rsq_adonis_source <- r_sq_ordered_adonis %>% pull(r_sq) %>% max
p_adonis_source <- r_sq_ordered_adonis %>% pull(p) %>% max

adonis_stats_dn101 <- read_tsv("../data/process/adonis_dn1-1.tsv")
dn1_rsq_sum <- adonis_stats_dn101 %>% filter(day == -1) %>% select(r_sq) %>% sum
d0_rsq_sum <-  adonis_stats_dn101 %>% filter(day == 0) %>% select(r_sq) %>% sum
d1_rsq_sum <-  adonis_stats_dn101 %>% filter(day == 1) %>% select(r_sq) %>% sum
p_adonis_intial <- adonis_stats_dn101 %>% select(p) %>% max 
p_adonis_min <- adonis_stats_dn101 %>% select(p) %>% min #All p values are same

source_adonis_dn1 <- read_tsv("../data/process/adonis_dn1_source.tsv")
adonis_schloss_rsq_sum <- source_adonis_dn1 %>% filter(source == "Schloss") %>% select(r_sq) %>% sum
p_adonis_schloss_max <- source_adonis_dn1 %>% filter(source == "Schloss") %>% select(p) %>% max
adonis_young_rsq_sum <- source_adonis_dn1 %>% filter(source == "Young") %>% select(r_sq) %>% sum
p_adonis_young_max <- source_adonis_dn1 %>% filter(source == "Young") %>% select(p) %>% max
```
Next, we compared the bacterial communities from the 6 colonies over the course of the experiment using principal coordinate analysis (PCoA) of the Theta YC distances. Permutational multivariate analysis of variance (PERMANOVA) analysis revealed colony source was the major factor explaining the observed variation across fecal communities (R^2^ = `r rsq_adonis_source`, *P* = `r p_adonis_source`) followed by interactions between cage and day of the experiment (Movie S1 and Table S5). Since, the majority of the perturbations happened over the initial days of the experiment, we decided to focus on the bacterial communities at baseline (day -1), after clindamycin treatment (day 0), and post-infection (day 1). For all 3 timepoints, source and the interaction with cage significantly explained most of the observed community variation (combined R^2^ = `r dn1_rsq_sum`, `r d0_rsq_sum`, `r d1_rsq_sum`, respectively; *P* = `r p_adonis_intial`; Fig. 3 and Table S6). We also compared baseline communities across the 2 experiments, and found experiment and cage significantly explained the observed variation only for the Schloss and Young lab mouse colonies (Fig. S2 and Table S7), suggesting vendor communities were relatively stable between the 2 experiments. Thus, mouse colony source was the factor that explained the most variation observed in the bacterial communities. Importantly with the exception of the 2 University colonies, the community of each source clustered apart from one another suggesting each community had a unique response to clindamycin treatment and *C. difficile* challenge.
```{r source_taxa}
#Number of different otus and families across sources at day -1, 0, and 1 timepoints
source_otus <- read_tsv("../data/process/otu_stats_dn1to1_combined.tsv")
source_otus_dn1 <- source_otus %>% filter(day == -1) %>% nrow()
source_families <- read_tsv("../data/process/family_stats_dn1to1_combined.tsv")
source_families_dn1 <- source_families %>% filter(day == -1) %>% nrow()
source_otus_d0 <- source_otus %>% filter(day == 0) %>% nrow()
source_families_d0 <- source_families %>% filter(day == 0) %>% nrow()
source_otus_d1 <- source_otus %>% filter(day == 1) %>% nrow()
source_families_d1 <- source_families %>% filter(day == 1) %>% nrow()

#Function to find which significant otus/genera/families are shared 
intersect_all <- function(a,b,...){
  Reduce(intersect, list(a,b,...))
}

#OTUs shared across baseline, post-clindamycin, and post-infection timepoints
source_o_dn1 <- source_otus %>% filter(day == -1) %>% pull(otu)
source_o_d0 <- source_otus %>% filter(day == 0) %>% pull(otu)
source_o_d1 <- source_otus %>% filter(day == 1) %>% pull(otu)
shared_source_otus_dn1to1 <- length(intersect_all(source_o_dn1, source_o_d0, source_o_d1))

#Families shared across baseline, post-clindamycin, and post-infection timepoints
source_f_dn1 <- source_families %>% filter(day == -1) %>% pull(family)
source_f_d0 <- source_families %>% filter(day == 0) %>% pull(family)
source_f_d1 <- source_families %>% filter(day == 1) %>% pull(family)
shared_source_families_dn1to1 <- length(intersect_all(source_f_dn1, source_f_d0, source_f_d1))
```
After finding differences at the community level, we next identified the taxa that varied across sources of mice over the initial days of the experiment. We examined bacterial relative abundances at the operational taxonomic unit (OTU) and family levels, expecting the number of differences to be reduced at the family level due to the nature of bacterial taxonomy [@Nguyen2015]. Focusing on the baseline communities first, there were `r source_otus_dn1` OTUs and only `r source_families_dn1` families (Table S8-9) with relative abundances that varied across colony sources. Clindamycin treatment reduced the number of taxa with relative abundances that varied across sources to `r source_otus_d0` OTUs and `r source_families_d0` families (Table S8-9). After *C. difficile* challenge, there were `r source_otus_d1` OTUs and `r source_families_d1` families (Table S8-9) with significantly different relative abundances across sources, as the communities started to recover from antibiotic treatment. In spite of the experimental perturbations that occurred during these 3 timepoints, there were `r shared_source_otus_dn1to1` OTUs (Fig 4A-C) and `r shared_source_families_dn1to1` families with relative abundances that consistently varied across colony sources (Fig. 4D-F). Importantly, some of the taxa that consistently varied across sources also shifted with clindamycin treatment. For example, *Proteus* increased after clindamycin treatment, but only in Taconic mice. *Enterococcus* was primarily found only in mice purchased from commercial vendors and also increased after clindamycin treatment. In summary, mouse bacterial communities significantly varied according to colony source throughout the course of the experiment and a consistent subset of bacterial taxa remained different across sources regardless of clindamycin and *C. difficile* challenge.

```{r clindamycin_taxa}
clindamycin_otus <- read_tsv("../data/process/otu_stats_dn1to0.tsv")
clindamycin_otus_sig <- clindamycin_otus %>% filter(p.value.adj < 0.05) %>% nrow()

clindamycin_families <- read_tsv("../data/process/family_stats_dn1to0.tsv")
clindamycin_families_sig <- clindamycin_families %>% filter(p.value.adj < 0.05) %>% nrow()
```
**Clindamycin treatment alters a subset of taxa that were found in all colony sources.** Although there were bacteria that consistently varied across colony sources, we also wanted to identify the bacteria that shifted after clindamycin treatment, regardless of colony source. By analyzing all mice that had sequence data from fecal samples collected at baseline and after clindamycin treatment, we identified `r clindamycin_otus_sig` OTUs and `r clindamycin_families_sig` families that were altered after clindamycin treatment (Fig. 5 and Table S10-11). Interestingly, when we compared the list of significant clindamycin impacted bacteria with the bacteria that consistently varied across groups over the initial 3 timepoints of our experiment, we found 3 OTUs (*Lachnospiraceae* (OTU 130), *Lactobacillus* (OTU 6), *Enterococcus* (OTU 23)) and 3 families (*Porphyromonadaceae*, *Enterococcaceae*, *Lachnospiraceae*) overlapped (Fig. 4, Fig. 5C-D). These findings demonstrate that clindamycin has a consistent impact on the fecal bacterial communities of mice from all colony sources and only a subset of the taxa also varied across colony sources.

```{r classification}
models_vs_random <- read_tsv("../data/process/classification_to_random.tsv")
max_p_vs_random <- models_vs_random %>% select(p.value) %>% max
median_otu_day0_AUROC <- models_vs_random %>% filter(model_name == "otu_day0") %>% pull(median)

padj_pairwise_comp <- read_tsv("../data/process/classification_model_pairwise_stats.tsv")
max_padj_otu_day0_comp <- padj_pairwise_comp %>% filter(str_detect(comparison, "otu_day0")) %>% select(p.value.adj) %>% max

#Function to find which significant otus/genera/families are shared 
intersect_all <- function(a,b,...){
  Reduce(intersect, list(a,b,...))
}

#Read in components of venn diagram script to be able to calculate comparisons:----
#OTUs identified in logistic regression classification models (20 OTUs with the highest ranking for each model)
interp_otus <- read_tsv("../data/process/combined_top20_otus_all_models.tsv")
interp_otus_dn1 <- interp_otus  %>% filter(model_input_day == -1) %>% pull(OTU)
interp_otus_d0 <- interp_otus  %>% filter(model_input_day == 0) %>% pull(OTU)
interp_otus_d1 <- interp_otus  %>% filter(model_input_day == 1) %>% pull(OTU)
interp_combined <- c(interp_otus_dn1, interp_otus_d0, interp_otus_d1)

#OTUs that vary across sources of mice on day -1, 0, or 1
sig_otu_source <- read_tsv("../data/process/otu_stats_dn1to1_combined.tsv")
`sig_otu_day-1` <- sig_otu_source %>% filter(day == -1) %>% pull(otu)
`sig_otu_day0` <- sig_otu_source %>% filter(day == 0) %>% pull(otu)
`sig_otu_day1` <- sig_otu_source %>% filter(day == 1) %>% pull(otu)
#OTUs that consistently vary across sources of mice on day-1, 0, or 1
shared_sig_otus_Dn1toD1 <- intersect_all(`sig_otu_day-1`, sig_otu_day0, sig_otu_day1)


#OTUs that were impacted by clindamycin treatment
`sig_otu_pairs` <- read_tsv("../data/process/otu_stats_dn1to0.tsv") %>% 
  filter(p.value.adj < 0.05) %>% pull(otu)

# Day -1 overlapping OTUs----
# Overlap between OTUs that varied across sources of mice on day -1 and top 20 taxa from logistic regression model based on day -1 community:
dayn1_and_interp_dn1_o <- intersect_all(`sig_otu_day-1`, `interp_otus_dn1`)
# Overlap between OTUs that were altered by clindamycin treatment and top 20 taxa from logistic regression model based on day -1 community:
paired_and_interp_dn1_o <- intersect_all(`sig_otu_pairs`, `interp_otus_dn1`)

# Day 0 overlapping OTUs----
# Overlap between OTUs that varied across sources of mice on day 0 and top 20 taxa from logistic regression model based on day 0 community:
day0_and_interp_d0_o <- intersect_all(`sig_otu_day0`, `interp_otus_d0`)
# Overlap between OTUs that were altered by clindamycin treatment and top 20 taxa from logistic regression model based on day 0 community:
paired_and_interp_d0_o <- intersect_all(`sig_otu_pairs`, `interp_otus_d0`)

# Day 1 overlapping OTUs----
# Overlap between OTUs that varied across sources of mice on day 1 and top 20 taxa from logistic regression model based on day 1 community:
day1_and_interp_d1_o <- intersect_all(`sig_otu_day1`, `interp_otus_d1`)
# Overlap between OTUs that were altered by clindamycin treatment and top 20 taxa from logistic regression model based on day 1 community:
paired_and_interp_d1_o <- intersect_all(`sig_otu_pairs`, `interp_otus_d1`)

#Combined overlapping OTUs based on day -1, 0, and 1 comparisons----
#Combined overlapping OTUs that varied by source and were in the top 20 taxa from at least one logistic regression model & remove any duplicates
dayn1to1_and_interp_combined <- unique(c(`dayn1_and_interp_dn1_o`, `day0_and_interp_d0_o`, `day1_and_interp_d1_o`))
#See which source OTUs show up as significant over multiple days
source_otus_w_duplicates <- c(`dayn1_and_interp_dn1_o`, `day0_and_interp_d0_o`, `day1_and_interp_d1_o`)
key_source_otus <- sort(source_otus_w_duplicates[duplicated(source_otus_w_duplicates)])
#Key source OTUs: "Bacteroides (OTU 2)", "Enterococcus (OTU 23)"
#Combined overlapping OTUs that were altered by clindamycin treatment and were in the top 20 taxa from at least one logistic regression model
paired_and_interp_combined <- unique(c(`paired_and_interp_dn1_o`, `paired_and_interp_d0_o`, `paired_and_interp_d1_o`))
#See which clindamycin-associated OTUs show up as significant over multiple days
clind_otus_w_duplicates <- c(`paired_and_interp_dn1_o`, `paired_and_interp_d0_o`, `paired_and_interp_d1_o`)
key_clind_otus <- sort(clind_otus_w_duplicates[duplicated(clind_otus_w_duplicates)])
#List of key OTUs from source and clindamycin varying OTUs:
key_otus <- unique(c(key_source_otus, key_clind_otus))
list_key_otus <- paste(key_otus, collapse= ", ") #Format to output in paper

#Families identified in logistic regression classification models (20 families with the highest ranking for each model)
intrep_families <- read_tsv("../data/process/combined_top20_families_all_models.tsv")
interp_families_dn1 <- intrep_families %>% filter(model_input_day == -1) %>% pull(family)
interp_families_d0 <- intrep_families %>% filter(model_input_day == 0) %>% pull(family)
interp_families_d1 <- intrep_families %>% filter(model_input_day == 1) %>% pull(family)
interp_families_combined <- c(interp_families_dn1, interp_families_d0, interp_families_d1)
#Families identified in logistic regression classification models (20 families with the highest ranking for each model)
intrep_families <- read_tsv("../data/process/combined_top20_families_all_models.tsv")
interp_families_dn1 <- intrep_families %>% filter(model_input_day == -1) %>% pull(family)
interp_families_d0 <- intrep_families %>% filter(model_input_day == 0) %>% pull(family)
interp_families_d1 <- intrep_families %>% filter(model_input_day == 1) %>% pull(family)
interp_families_combined <- c(interp_families_dn1, interp_families_d0, interp_families_d1)

#Families that vary across sources of mice on day -1, 0, or 1.
sig_family_source <- read_tsv("../data/process/family_stats_dn1to1_combined.tsv")
`sig_family_day-1` <- sig_family_source %>% filter(day == -1) %>% pull(family)
`sig_family_day0` <- sig_family_source %>% filter(day == 0) %>% pull(family)
`sig_family_day1` <- sig_family_source %>% filter(day == 1) %>% pull(family)
#Families that consistently vary across sources of mice on day-1, 0, or 1
shared_sig_families_Dn1toD1 <- intersect_all(`sig_family_day-1`, sig_family_day0, sig_family_day1)

#Families that were impacted by clindamycin treatment
`sig_family_pairs` <- read_tsv("../data/process/family_stats_dn1to0.tsv") %>% 
  filter(p.value.adj < 0.05) %>% pull(family)

# Day -1 overlapping families----
# Overlap between families that varied across sources of mice on day -1 and top 20 taxa from logistic regression model based on day -1 community:
dayn1_and_interp_dn1_f <- intersect_all(`sig_family_day-1`, `interp_families_dn1`)
# Overlap between families that were altered by clindamycin treatment and top 20 taxa from logistic regression model based on day -1 community:
paired_and_interp_dn1_f <- intersect_all(`sig_family_pairs`, `interp_families_dn1`)

# Day 0 overlapping families----
# Overlap between families that varied across sources of mice on day 0 and top 20 taxa from logistic regression model based on day 0 community:
day0_and_interp_d0_f <- intersect_all(`sig_family_day0`, `interp_families_d0`)
# Overlap between families that were altered by clindamycin treatment and top 20 taxa from logistic regression model based on day 0 community:
paired_and_interp_d0_f <- intersect_all(`sig_family_pairs`, `interp_families_d0`)

# Day 1 overlapping families----
# Overlap between families that varied across sources of mice on day 1 and top 20 taxa from logistic regression model based on day 1 community:
day1_and_interp_d1_f <- intersect_all(`sig_family_day1`, `interp_families_d1`)
# Overlap between families that were altered by clindamycin treatment and top 20 taxa from logistic regression model based on day 1 community:
paired_and_interp_d1_f <- intersect_all(`sig_family_pairs`, `interp_families_d1`)

#Combined overlapping families based on day -1, 0, and 1 comparisons----
#Combined overlapping families that varied by source and were in the top 20 taxa from at least one logistic regression model & remove any duplicates
dayn1to1_and_interp_combined_f <- unique(c(`dayn1_and_interp_dn1_f`, `day0_and_interp_d0_f`, day1_and_interp_d1_f))
#See which source families show up as significant over multiple days
source_families_w_duplicates <- c(`dayn1_and_interp_dn1_f`, `day0_and_interp_d0_f`, day1_and_interp_d1_f)
key_source_families <- sort(unique(source_families_w_duplicates[duplicated(source_families_w_duplicates)]))
#Key source families: "Bacteroidaceae" -seen across 3 days, "Deferribacteraceae", "Enterococcaceae", "Lachnospiraceae"
#Combined overlapping families that were altered by clindamycin treatment and were in the top 20 taxa from at least one logistic regression model
paired_and_interp_combined_f <- unique(c(`paired_and_interp_dn1_f`,`paired_and_interp_d0_f`, paired_and_interp_d1_f))
#See which clindamycin-associated families show up as significant over multiple days
clind_familes_w_duplicates <- c(`paired_and_interp_dn1_f`,`paired_and_interp_d0_f`, paired_and_interp_d1_f)
key_clind_families <- sort(unique(clind_familes_w_duplicates[duplicated(clind_familes_w_duplicates)]))
#List of key families from source and clindamycin varying families:
key_families <- unique(c(key_source_families, key_clind_families))
list_key_families <- paste(key_families, collapse= ", ") #Format to output in paper

#Venn diagram calculations for OTUs----
venn_otus_source <- length(dayn1to1_and_interp_combined)
venn_otus_intersect <- length(intersect(dayn1to1_and_interp_combined, paired_and_interp_combined))
venn_otus_clind <-  length(paired_and_interp_combined)
#Venn diagram calculations for families
venn_families_source <- length(dayn1to1_and_interp_combined_f)
venn_families_intersect <- length(intersect(dayn1to1_and_interp_combined_f, paired_and_interp_combined_f))
venn_families_clind <-  length(paired_and_interp_combined_f)
```
**Source-specific and clindamycin impacted bacteria distinguish *C. difficile* colonization status in mice.** After identifying taxa that varied by colony source, changed after clindamycin treatment, or both, we next wanted to determine which taxa were influencing the variation in *C. difficile* colonization status at day 7 (Fig. 1D). We trained L2-regularized logistic regression models with input bacterial community data from the baseline, post-clindamycin, and post-infection timepoints of the experiment to predict *C. difficile* colonization status on day 7 (Fig. S3). All models were better at predicting *C. difficile* colonization status on day 7 than random chance (all *P* $\le$ `r format(max_p_vs_random, scientific = TRUE, digits = 2)`; Table S12), however the models trained with OTU level data generally performed better than those trained with family level data with the expception of the models based on the post-infection (day 1) communities (Fig. 6). Interestingly, the model based on the post-clindamycin (day 0) community OTU data performed significantly better than all other models with an AUROC of `r median_otu_day0_AUROC` (*P*~FDR~ $\le$ `r format(max_padj_otu_day0_comp, scientific = TRUE, digits = 2)` for pairwise comparisons; Table S13). Thus, we were able to use community bacterial relative abundance data alone to differentiate mice that were still colonized with *C. difficile* on day 7 from those mice that had cleared the *C. difficile*. Interestingly, the model built with OTU relative abundance data post-clindamycin treatment had the best performance, suggesting how the bacterial community responds to clindamycin treatment has the greatest influence on subsequent *C. difficile* colonization dynamics.

Next, to examine the bacteria that were driving each model’s performance, we pulled out the top 20 taxa that had the highest absolute feature weights in each of the 6 models (Table S14-15). The top 20 taxa from each model were compared to the list of taxa that varied across colony source (Fig. 4 and Table S8-9) at the same timepoint and the taxa that were altered by clindamycin treatment (Fig. 5 and Table S10-11). We found a subset of OTUs and families that were important to the model and overlapped with bacteria that varied by either source, clindamycin treatment, or both (Fig. S4, S5, S6). Combining the overall results for the 3 OTU models identified `r venn_otus_source` OTUs associated with source, `r venn_otus_clind` OTUs associated with clindamycin treatment, and `r venn_otus_intersect` OTUs associated with both (Fig. 7A). Combining the overall results for the 3 family models identified `r venn_families_source` families associated with source, `r venn_families_clind` families associated with clindamycin treatment and `r venn_families_intersect` families associated with both (Fig. 7B). Several OTUs (*`r list_key_otus`*) and families (*`r list_key_families`*) appeared across at least 2 models, so we examined how the relative abundances of these key taxa varied over the course of the experiment (Fig. 8 and Fig. S7). Throughout the experiment, there was at least 1 timepoint where relative abundances of these taxa significantly varied across sources (Table S16-17). Interestingly, there were no taxa that emerged as consistently enriched or depleted in Jackson, Taconic, and Envigo mice that were colonized for longer with *C. difficile* 630, suggesting multiple bacteria influence the time needed to clear the infection. Together, these results suggest the initial bacterial communities and their responses to clindamycin have a large influence on the time needed to clear *C. difficile*.

## Discussion

By examining the *C. difficile* colonization dynamics within mice from 6 different colony sources after perturbing the microbiota with clindamycin treatment, we were able to identify bacterial taxa that were unique to sources throughout the experiment as well as taxa that were universally impacted by clindamycin. We built L2 logistic regression models with baseline, post-clindamycin treatment, and post-infection fecal community data that successfully predicted *C. difficile* colonization status 7 days after infection better than random chance. We identified *`r list_key_otus`*, *`r list_key_families`* (Fig. 8, Fig. S7) as candidate bacteria within these communities that were influencing variation in *C. difficile* colonization dynamics since these bacteria were all important in the logistic regression models and varied by colony source, were impacted by clindamycin treatment, or both. Overall, our results demonstrate clindamycin is sufficient to render mice from multiple sources susceptible to CDI and only a subset of the interindividual microbiota variation across mice from different sources was associated with the time needed to clear *C. difficile*. 

Other groups have taken similar approaches by using mice from multiple colony sources to identify bacteria that either promote colonization resistance or increase susceptibility to enteric infections [@Velazquez2019; @EtienneMesmin2017; @Osbelt2020; @Lai2020; @Thiemann2017; @Rolig2013; @Ge2018]. For example, in the context of *Salmonella* infections, *Enterobacteriaceae* and segmented filamentous bacteria have emerged as protective [@Velazquez2019; @Lai2020]. A previous study with *C. difficile* identified an endogenous protective *C. difficile* strain LEM1 that bloomed after antibiotic treatment in mice from Jackson or Charles River Laboratories, but not Taconic that protected mice against the more toxigenic *C. difficile* VPI10463 [@EtienneMesmin2017]. Given that we ordered mice from the same vendors, we checked all mice for endogenous *C. difficile* by plating stool samples that were collected after clindamycin treatment. However, we did not identify any endogenous *C. difficile* strains prior to challenge, suggesting there were no endogenous protective strains in the mice we received and other bacterial taxa were mediating the variation in *C. difficile* colonization across sources. Although all mice were susceptible to *C. difficile* colonization, by following colonization over time we found Jackson, Taconic, and Envigo mice remained colonized for longer. We identified a subset of bacteria that were important in predicting whether a mouse was still colonized with *C. difficile* status 7 days post-infection. These results suggest a subset of the bacterial community is responsible for determining the length of time needed to clear *C. difficile* colonization.

In the past variation between different CDI mouse model studies have been attributed to intestinal microbiome differences in mice across different institutional environments. For example, groups using the same clindamycin treatment and C57BL/6 mice had different *C. difficile* outcomes, one having sustained colonization [@Buffie2011], while the other had transient [@Reeves2011]. Baseline differences in the microbiota composition have been hypothesized to partially explain the differences in colonization outcomes and overall susceptibility to *C. difficile* after treatment with the same antibiotic [@Lawley2013; @Hutton2014]. We have shown that mice from 6 different sources were all susceptible to *C. difficile* 630, suggesting the microbiota influences *C. difficile* clearance more than susceptibility. Fortunately, the bacterial perturbations induced by clindamycin treatment have been well characterized and our findings agree with previous CDI mouse model work demonstrating *Enterococcus* and *Enterobacteriaceae* were associated with *C. difficile* susceptibility and *Porpyhromonadaceae*, *Lachnospiraceae*, *Ruminococcaceae*, and *Turicibacter* were associated with resistance [@Schubert2015; @Jenior2018; @Tomkovich2019; @Buffie2011; @Buffie2014; @Lawley2009; @Lawley2012; @Jump2014]. While we have demonstrated that susceptibility is uniform across vendors after clindamycin treatment, there could be different outcomes for either susceptibility or clearance in the case of other antibiotic treatments. The *C. difficile* strain used could also be contributing to the variation in *C. difficile* outcomes seen across different groups [@Lawley2012]. We found the time needed to naturally clear *C. difficile* varied across sources of mice implying that at least in the context of the same perturbation, microbiota differences seemed to influence infection outcome more than susceptibility. More importantly, we were able to narrow down from all the variation observed across colony sources to a subset of bacterial taxa that were also important for predicting *C. difficile* colonization status 7 days post-infection.

Our approach successfully increased the diversity of murine bacterial communities tested in our clindamycin *C. difficile* model. One alternative approach that has been used in some CDI studies [@NagaoKitamoto2020; @Battaglioli2018; @Robinson2014; @Collins2015; @Collins2018; @Hryckowian2018] is to use mice associated with human microbiotas. However, a major caveat to this method is the substantial loss of human microbiota community members upon transfer to mice [@Fouladi2020; @Walter2020]. Additionally with the exception of 2 recent studies [@NagaoKitamoto2020; @Battaglioli2018], most of the CDI mouse model studies to date associated mice with just 1 types of human microbiota either from a single donor or a single pool from multiple donors [@Robinson2014; @Collins2015; @Collins2018; @Hryckowian2018], which does not aid in the goal of figuring out how a variety of unique microbiotas influence susceptiblity to CDIs and adverse outcomes. Encouragingly, decreased *Bifidobacterium*, *Porphyromonas*, *Ruminococcaceae* and *Lachnospiraceae* and increased *Enterobacteriaceae*, *Enterococcus*, *Lactobacillus*, and *Proteus* have all been associated with human CDIs [@Mancabelli2017, @Duvallet2017] and were well represented in our study, suggesting most of the vendors are suitable for gaining insights into microbiota associated factors influencing *C. difficile* colonization and infections in humans. An important exception was *Enterococcus*, which was primarily absent from the mice from University of Michigan colonies and *Proteus*, which was only found in Taconic mice. Importantly, the fact that some CDI associated bacteria were only found in a subset of mice has important implications for future CDI mouse model studies. 

There are several limitations to our work. The microbiome is composed of viruses, fungi, and parasites in addition to bacteria, and these non-bacterial members can also vary across mouse vendors [@Rasmussen2019; @Mims2020]. While our study focused solely on the bacterial portion, viruses and fungi have also begun to be implicated in the context of CDIs or FMT treatments for recurrent CDIs [@Markey2018; @Stewart2019; @Ott2017; @Zuo2017; @Zuo2018]. Beyond community composition, the metabolic function of the microbiota also has a CDI signature [@Robinson2019; @Jenior2017; @Fletcher2018; @Jump2014] and can vary across mice from different sources [@Xiao2015]. For example, microbial metabolites, particularly secondary bile acids and butyrate production, have been implicated as important contributors to *C. difficile* resistance [@Buffie2014; @Lawley2012]. Although, we only looked at composition, *Ruminococcaceae* and *Lachnospiraceae* both emerged as important taxa for classifying day 7 *C. difficile* colonization status and metagenomes from these bacteria have been shown to contain the bile acid-inducible gene cluster necessary for secondary bile acid formation and ability to produce butyrate [@Vital2019; @Collins2015]. Interestingly, butyrate has previously been shown to vary across vendors and mediates resistance to *Citrobacter rodentium* infection, a model of enterohemorrhagic and enteropathogenic *Escherichia coli* infections [@Osbelt2020]. Evidence for immunological toning differences in IgA and Th17 cells across mice from different vendors have also been documented and [@Fransen2015; @Ivanov2009], may also influence response to CDI, particularly in the context of severe CDIs [@Azrad2018; @Saleh2019]. The outcome after *C. difficile* exposure depends on a multitude of factors, including age, diet, and immunity; all of which are also influenced by the microbiota. We have demonstrated that the ways different baseline microbiotas from different mouse colony sources respond to clindamycin treatment influences the length of time mice remained colonized with *C. difficile* 630. For those interested in dissecting the contribution of the microbiome to *C. difficile* pathogenesis and treatments, using multiple sources of mice may yield more insights than a single model alone. Furthermore, for studies wanting to examine the interplay between a particular bacterial taxon such as *Enterococcus* and *C. difficile*, these results could serve as a resource for selecting which mice to order to address the question. 

\newpage

## Acknowledgements
This work was supported by the National Institutes of Health (U01AI124255). ST was supported by the Michigan Institute for Clincial and Health Research Postdoctoral Translation Scholars Program (UL1TR002240 and KL2TR002241). We thank members of the Schloss lab for feedback on planning the experiments and data presentation, as well as code tutorials and feedback through Code club. In particular, we want to thank Begum Topçuoğlu for help with implementing logistic regression models using her [pipeline](https://github.com/SchlossLab/ML_pipeline_microbiome) and Ana Taylor for help with some media preparation and sample collection. We also thank members of Vincent Young’s lab, particularly  Kimberly Vendrov, for guidance with the *C. difficile* infection mouse model and donating the mice. We also want to thank the Unit for Laboratory Animal Medicine at the University of Michigan for maintaining our mouse colony and providing the institutional support for our mouse experiments. Finally, we thank Kwi Kim, Austin Campbell, and Kimberly Vendrov for their help in maintaining the Schloss lab’s anaerobic chamber.

\newpage

## Materials and Methods

**(i) Animals.** All experiments were approved by the University of Michigan Animal Care and Use Committee (IACUC) under protocol number PRO00006983. Female C57BL/7 mice were obtained from 6 different colony sources: The Jackson Laboratory, Charles River Laboratories, Taconic Biosciences, Envigo, and two colonies at the University of Michigan (the Schloss lab colony and the Young lab colony). The Young lab colony was originally established with mice purchased from Jackson, and the Schloss lab colony was later founded with mice donated from the Young lab. The 4 groups of mice purchased from vendors were allowed to acclimate to the University of Michigan mouse facility for 13 days prior to starting the experiment. At least 4 female mice (age 5-10 weeks) were obtained per source and mice from the same source were primarily housed at a density of 2 mice per cage. The experiment was repeated once, approximately 3 months after the start of the first experiment.

**(ii) Antibiotic treatment.** After the 13-day acclimation period and 1 day prior to challenge (Fig. 1A), all mice received 10 mg/kg clindamycin (filter sterilized through a 0.22 micron syringe filter prior to administration) via intraperitoneal injection.

**(iii) *C. difficile* infection model.** Mice were challenged with 10^3^ spores of *C. difficile* strain 630 via oral gavage post-infection 1 day after clindamycin treatment as described previously [@Jenior2018]. Mice weights and stool samples were taken daily through 9 days post-challenge. Collected stool was split for *C. difficile* CFU quantification and 16S rRNA sequencing analysis. *C. difficile* quantification stool samples were transferred to the anaerobic chamber, serially diluted in PBS, plated on taurocholate-cycloserine-cefoxitin-fructose agar (TCCFA) plates, and counted after 24 hours of incubation at 37&deg;C under anaerobic conditions. A sample from the day 0 timepoint (post-clindamycin and prior to *C. difficile* challenge) was also plated on TCCFA to ensure mice were not already colonized with *C. difficile* prior to infection. There were 3 deaths recorded over the course of the experiment, 1 Taconic mouse died prior to *C. difficile* challenge and 1 Jackson and 1 Envigo mouse died between 1- and 3-days post-infection. Mice were categorized as cleared when no *C. difficile* was detected in the first serial dilution (limit of detection: 100 CFU). Stool samples for 16S rRNA sequencing were snap frozen in liquid nitrogen and stored at -80&deg;C until DNA extraction.

**(iv) 16S rRNA sequencing.** DNA was extracted from -80&deg;C stored stool samples using the DNeasy Powersoil HTP 96 kit (Qiagen) and an EpMotion 5075 automated pipetting system (Eppendorf). The V4 region was amplified for 16S rRNA with the AccuPrime Pfx DNA polymerase (Thermo Fisher Scientific) using custom barcoded primers, as previously described [@Kozich2013]. The ZymoBIOMICS microbial community DNA standards was used as a mock community control [@Sze2019] and water was used as a negative control per 96-well extraction plate. The PCR amplicons were cleaned up and normalized with the SequalPrep normalization plate kit (Thermo Fisher Scientific). Amplicons were pooled and quantified with the KAPA library quantification kit (KAPA biosystems), prior to sequencing using the MiSeq system (Illumina).

**(v) 16S rRNA gene sequence analysis.** mothur (v. 1.43) was used to process all sequences [@Schloss2009] with a previously published protocol [@Kozich2013]. Reads were combined and aligned with the SILVA reference database [@Quast2012]. Chimeras were removed with the VSEARCH algorithm and taxonomic assignment was completed with a modified version (v16) of the Ribosomal Database Project reference database (v11.5) [@Cole2013] with an 80% cutoff. Operational taxonomic units (OTUs) were assigned with a 97% similarity threshold using the opticlust algorithm [@Westcott2017]. To account for uneven sequencing across samples, samples were rarefied to 5,437 sequences 1,000 times for alpha and beta diversity analyses. PCoAs were generated based on Theta YC distances. Permutational multivariate analysis of variance (PERMANOVA) was performed on mothur-generated Theta YC distance matrices with the adonis function in the vegan package [@Vegan2018] in R [@r_citation_2018].

**(vi) Classification model training and evaluation.** Models were generated based on mice that were categorized as either cleared or colonized 7 days post-infection and had sequencing data from the baseline (day -1), post-clindamycin (day 0), and post-infection (day 1) timepoints of the experiment. Input bacterial community relative abundance data at either the OTU or family level from the baseline, post-clindamycin, and post-infection timepoints was used to generate 6 classification models that predicted *C. difficile* colonization status 7 days post-infection. The L2-regularized logistic regression models were trained and tested using the caret package [@Kuhn2008] in R as previously described [@Topcuoglu2019] with the exception that we used 60% training and 40% testing data splits for the cross-validation of the training data to select the best cost hyperparameter and the testing of the held out test data to measure model performance. The modified training/testing ratio was selected to accommodate the small number of samples in the dataset. Code was modified from https://github.com/SchlossLab/ML_pipeline_microbiome to update the classification outcomes and change the data split ratios. The modified repository to regenerate this analysis is available at https://github.com/tomkoset/ML_pipeline_microbiome.

**(vii) Statistical analysis.** All statistical tests were performed in R (v `r paste(version$major, ".", version$minor, sep = "")`) [@r_citation_2018]. The Kruskal-Wallis test was used to analyze differences in *C. difficile* CFU, mouse weight change, and alpha diversity across vendors with a Benjamini-Hochberg correction for testing multiple timepoints, followed by pairwise Wilcoxon comparisons with Benjamini-Hochberg correction. For taxonomic analysis and generation of logistic regression model input data, *C. difficile* (OTU 20) was removed. Bacterial relative abundances that varied across sources at the OTU and family taxonomic levels were identified with the Kruskal-Wallis test with Benjamini-Hochberg correction for testing all identified taxa at each level, followed by pairwise Wilcoxon comparisons with Benjamini-Hochberg correction. Taxa impacted by clindamycin treatment were identified using the Wilcoxon signed rank test with matched pairs of mice samples for day -1 and day 0. To determine whether classification models had better performance (test AUROCs) than random chance (0.5), we used the one-sample Wilcoxon signed rank test. To examine whether there was an overall difference in predictive performance across the 6 classification models we used the Kruskal-Wallis test followed by pairwise Wilcoxan comparisons with Benjamini-Hochberg correction for multiple hypothesis testing. The tidyverse package was used to wrangle and graph data (v `r paste(packageVersion("tidyverse")[1])`) [@Tidyverse2019].

**(viii) Code availability.** Code for all data analysis and generating this manuscript is available at https://github.com/SchlossLab/Tomkovich_vendor_difs_XXXX_2020.

**(ix) Data availability.** The 16S rRNA sequencing data have been deposited in the National Center for Biotechnology Information Sequence Read Archive (BioProject Accession no. PRJNA608529).

\newpage

## References

<div id="refs"></div>

\newpage

## Figures
```{r sample_size}
#figure out in functions.R and export results as a tsv to read in
analysis_n <- read_tsv("../data/process/processed_metadata.tsv")

#Initial N for each source from the 2 experiments combined:
schloss_i <- analysis_n %>% filter(day == -1 & vendor == "Schloss") %>% nrow()
young_i <- analysis_n %>% filter(day == -1 & vendor == "Young") %>% nrow()
jackson_i <- analysis_n %>% filter(day == -1 & vendor == "Jackson") %>% nrow()
cr_i <- analysis_n %>% filter(day == -1 & vendor == "Charles River") %>% nrow()
taconic_i <- analysis_n %>% filter(day == -1 & vendor == "Taconic") %>% nrow()
envigo_i <- analysis_n %>% filter(day == -1 & vendor == "Envigo") %>% nrow()

#Range in N for C. difficile CFU quantification:
cfu_n_by_day <- analysis_n %>% 
  filter(!is.na(cfu)) %>% #remove rows with NAs
  group_by(day) %>% count() %>% arrange(n) %>% ungroup()
min_cfu_n <- cfu_n_by_day %>% select(n) %>% min
max_cfu_n <- cfu_n_by_day %>% select(n) %>% max

#Range in N for mouse weight change data over the course of the experiment:
weight_n_by_day <- analysis_n %>% 
  filter(!is.na(weight)) %>% #remove rows with NAs
  group_by(day) %>% count() %>% arrange(n) %>% ungroup()
min_weight_n <- weight_n_by_day %>% select(n) %>% min
max_weight_n <- weight_n_by_day %>% select(n) %>% max

#C. difficile day 7 colonization status N range across sources:
day7_status_n <- analysis_n %>% 
  filter(!is.na(clearance_status_d7)) %>% #remove rows with NAs
  filter(day == 7) %>% 
  group_by(vendor) %>% count() %>% arrange(n) %>% ungroup
day7_status_n_min <- day7_status_n %>% select(n) %>% min
day7_status_n_max <-day7_status_n %>% select(n) %>% max

#N for mice with sequencing data on both day -1 and day 0 (used to determine taxa that were altered by clindamycin treatment)
n_paired_for_clind <- read_tsv("../data/process/vendors.diversity.summary") %>% #read in sequencing data & join to metadata
  filter(method == "ave") %>% 
  select(group, sobs, shannon, invsimpson, coverage) %>% 
  inner_join(metadata, by = c("group" = "id")) %>% 
  filter(day == -1 | day == 0) %>% 
  filter(duplicated(mouse_id)) %>% 
  nrow()
```
\includegraphics{figure_1.pdf}
**Figure 1. Clindamycin is sufficient to promote *C. difficile* colonization regardless of colony source, but clearance time varies across sources of C57BL/6 mice.** A. Setup of the experimental timeline. Mice for the experiments were obtained from 6 different sources: the Schloss (N = `r schloss_i`) and Young lab (N = `r young_i`) colonies at the University of Michigan, the Jackson Laboratory (N = `r jackson_i`), Charles River Laboratory (N = `r cr_i`), Taconic Biosciences (N = `r taconic_i`), and Envigo (N = `r envigo_i`). All mice were administered 10 mg/kg clindamycin intraperitoneally (IP) 1 day before challenge with *C. difficile* 630 spores on day 0. Mice were weighed and feces was collected daily through the end of the experiment (9 days post-infection). Note: 3 mice died during course of experiment. 1 Taconic mouse prior to infection and 1 Jackson and 1 Envigo mouse between 1- and 3-days post-infection. B. *C. difficile* CFU/gram stool measured over time (N = `r min_cfu_n`-`r max_cfu_n` mice per timepoint) via serial dilutions. The black line represents the limit of detection for the first serial dilution. CFU quantification data was not available for each mouse due to early deaths, stool sampling difficulties, and not plating all of the seriral dilutions. C. Mouse weight change measured in grams over time (N = `r min_weight_n`-`r max_weight_n` mice per timepoint), all mice were normalized to the weight recorded 1 day before infection. For B-C: timepoints where differences across sources of mice were statistically significant by Kruskal-Wallis test with Benjamini-Hochberg correction for testing across multiple days (Table S1 and Table S2) are reflected by the asterisk(s) above each timepoint (\*\, P < 0.05, \*\*\, P < 0.01, \*\*\*\, P < 0.001). Lines represent the mean for each source and circles represent individual mice. D. *C. difficile* CFU status on Day 7 within each mouse colony source. Mice were classified as colonized or cleared (not detectable at the limit of detection of 100 CFU). N = `r day7_status_n_min`-`r day7_status_n_max` mice per group. 

\newpage
\includegraphics{figure_2.pdf}
**Figure 2. Differences in microbial richness and diversity across mouse colony sources emerge after clindamycin treatment and infection.** A-B. Boxplots of the number of observed OTUs and Shannon diversity index values at baseline: day -1 (A-B), after clindamycin: day 0 (C-D) and post-infection: day 1 (E-F) timepoints of the experiment. Each circle represents the value for a stool sample from an individual mouse. Data were analyzed by Kruskal-Wallis test with Benjamini-Hochberg correction for testing each day of the experiment and the adjusted *P* value was < 0.05 for all panels except for B (Table S3). Significant *P* values from the pairwise Wilcoxon comparisons between sources with Benjamini-Hochberg correction are shown (Table S4).

\newpage
\includegraphics{figure_3.pdf}
**Figure 3. Mouse colony source is the variable that explains most of the variation observed in the baseline, post-clindamycin, and post-infection bacterial communities.** A-C. Principal Coordinates Analysis of Theta YC distances from stools collected at baseline (A), post-clindamycin (B), and post-infection (C) timepoints of the experiment. Each circle represents a stool sample from an individual mouse. PERMANOVA analysis demonstrated that source and the interaction between source and cage explained most of the variation observed in the baseline (combined R^2^ = `r dn1_rsq_sum`), post-clindamycin (combined R^2^ = `r d0_rsq_sum`), and post-infection (combined R^2^ = `r d1_rsq_sum`) communities (all *P* = `r p_adonis_intial`, see Table S6).

\newpage
\includegraphics{figure_4.pdf}
**Figure 4. A subset of bacterial consistently vary across mouse colony sources despite clindamycin perturbation and *C. difficile* challenge.** A-C. Boxplots of the relative abundances for the `r shared_source_otus_dn1to1` OTUs that consistently varied across sources of mice at the baseline (A), post-clindamycin (B), and post-infection (C) timepoints of the experiment. D-F. Boxplots of the relative abundances for the `r shared_source_families_dn1to1` families that consistently varied across sources of mice at the baseline (D), post-clindamycin (E), and post-infection (F) timepoints of the experiment. For each timepoint bacteria with differential relative abundances across sources of mice were identified by Kruskal-Wallis test at the family and OTU level with Benjamini-Hochberg correction for testing all identified taxa at the respective level (Table S8-9). The grey vertical line indicates the limit of detection for A-F.

\newpage
\includegraphics{figure_5.pdf}
**Figure 5. Clindamycin treatment has the same effects on a subset of taxa regardless of colony source.** A-B. Boxplots of the top 10 most significant (adjusted *P* value < 0.05) OTUs with relative abundances that changed post clindamycin treatment. C-D. Boxplots of the top 10 most significant families with relative abundances that changed post clindamycin treatment. Data were analyzed by Wilcoxon signed rank test limited to mice that had paired sequence data for day -1 and 0 (N = `r n_paired_for_clind`). Tests were performed at the OTU and family levels with Benjamini-Hochberg correction for testing all identified OTUs and families. See Table S10-11 for complete list of OTUs and families significantly impacted by clindamycin treatment. The grey vertical line indicates the limit of detection for A-D.

\newpage
\includegraphics{figure_6.pdf}
**Figure 6. Bacterial community composition before, after clindamycin perturbation, and post-infection can predict *C. difficile* colonization status 7 days post-challenge.** A-B. Logistic regression classification model AUROCs to predict *C. difficile* CFU on D7 (Fig. 1D, Fig. S3) based on the community relative abundances at baseline (day -1), post-clindamycin (day 0), and post-infection (day 1) at either the OTU (A) or family (B) level. All models performed better than random chance (AUROC = 0.5), see (all *P* $\le$ `r format(max_p_vs_random, scientific = TRUE, digits = 2)`; Table S12) and the model built with post-clindamycin treated bacterial OTU relative abundances had the best performance ((*P*~FDR~ $\le$ `r format(max_padj_otu_day0_comp, scientific = TRUE, digits = 2)` for pairwise comparisons; Table S13). A List of the 20 taxa that were ranked as most important to each model are listed in Table S14-15.

\newpage
\includegraphics{figure_7.pdf}
**Figure 7. Key taxa that influence *C. difficile* colonization status vary by mouse colony source, change after clindamycin treatment, or both.** A-B. Venn diagrams that combine Fig. S4-6 summaries of taxa that were important to the day -1, 0, and 1 classification models (Table S14-15) and either overlapped with taxa that varied across vendors at the same timepoint, were impacted by clindamycin treatment, or both. See Fig. S4-6 for separate comparisons of taxa from the day -1, 0, and 1 classification models. Bold taxa signify bacteria that were important to more than 1 classification model.

\newpage
\includegraphics{figure_8.pdf}
**Figure 8: Key OTUs vary across sources throughout the experiment.** A-C. Relative abundances of bold OTUs from Fig. 7A  that were important for at least two classification models are shown over time. A. *Enterococcus* (OTU 23), which significantly varied across sources and was impacted by clindamycin treatment. B. *Bacteroides* (OTU 2), which varied across sources throughout the experiment. C. *Enterobacteriaceae* (OTU 1) and *Porphyromonadaceae* (OTU 7) were significantly impacted by clindamycin treatment and examining relative abundance dynamics over the course of the experiment indicated timepoints where relative abundances also significantly varied across sources of mice. Each point represents the relative abundance of an individual mouse stool sample and colored lines represent the mean relative abundances for each source of mice. The gray horizontal line represents the limit of detection. Timepoints where differences across sources of mice were statistically significant by Kruskal-Wallis test with Benjamini-Hochberg correction for testing across multiple days (Table S16) are identified by the asterisk(s) above each timepoint (\*\, P < 0.05, \*\*\, P < 0.01, \*\*\*\, P < 0.001).

\newpage
\includegraphics{figure_S1.pdf}
**Figure S1. *C. difficile* CFU variation across vendors is similar between the 2 experiments.** (A-B). *C. difficile* CFU/gram of stool quantification over time for experiment 1 (A) and 2 (B). Experiments were conducted approximately 3 months apart. Lines represent the mean CFU for each source and circles represent individual mice and the black line represents the limit of detection.

\newpage
\includegraphics{figure_S2.pdf}
**Figure S2. Only bacterial communities from University of Michigan mice significantly vary between experiments.** A-F. PCoA of Theta YC distances for the baseline fecal bacterial communities within each source of mice. Each symbol represents a stool sample from an individual mouse with color corresponding to experiment and shape representing cage mates. PERMANOVA was performed within each group to examine the contributions of experiment and cage to observed variation. Experiment number and cage only significantly explained observed variation for mice from the Schloss (combined R^2^ = `r adonis_schloss_rsq_sum`; *P* $\le$ `r p_adonis_schloss_max`) and Young (combined R^2^ = `r adonis_young_rsq_sum`; *P* $\le$ `r p_adonis_young_max`) lab colonies (Table S7).

\newpage
\includegraphics{figure_S3.pdf}
**Figure S3. *C. difficile* colonization status 7 days post-infection varies across mice.** A. Bar graph visualizations of overall day 7 C. difficile colonization status that were used as classification outcomes to build logistic regression models. Mice were classified as colonized or cleared (not detectable at the limit of detection of 100 CFU) based on CFU g/stool data from 7 days post-infection. B-D. Classification input data used for models based on day -1 (B), day 0 (C), and day 1 (D) community relative abundance data. Only mice with day 7 *C. difficile* status and community relative abundance data for each respective timepoint were used as input data for the classification models.

\newpage
\includegraphics{figure_S4.pdf}
**Figure S4. Key taxa from classification model based on baseline community data vary by mouse colony source, clindamycin treatment, or both.** A-B. Venn diagrams of top 20 important OTUs (A) and families (B) from day -1 classification models (Table S14-15) that overlapped with taxa that varied across vendors at baseline, were impacted by clindamycin treatment, or both. Bold taxa signify bacteria that were important to more than 1 classification model.

\newpage
\includegraphics{figure_S5.pdf}
**Figure S5. Key taxa from classification model based on post-clindamycin treatment community data vary by mouse colony source, clindamycin treatment, or both.** A-B. Venn diagrams of top 20 important OTUs (A) and families (B) from day 0 classification models (Table S14-15) that overlapped with taxa that still varied across vendors after clindamycin, were impacted by clindamycin treatment, or both. Bold taxa signify bacteria that were important to more than 1 classification model.

\newpage
\includegraphics{figure_S6.pdf}
**Figure S6. Key taxa from classification model based on post-infection community data vary by mouse colony source, clindamycin treatment, or both.** A-B. Venn diagrams of top 20 important OTUs (A) and families (B) from day 1 classification models (Table S14-15) that overlapped with taxa that varied across vendors post-infection, were impacted by clindamycin treatment, or both. Bold taxa signify bacteria that were important to more than 1 classification model.

\newpage
\includegraphics{figure_S7.pdf}
**Figure S7. Key families vary across sources throughout experiment.** Relative abundances of bold families from Fig. 7B that were important for at least two classification models are shown over time. A. *Enterococcaceae* and *Lachnospiraceae*, which significantly varied across sources and were impacted by clindamycin treatment. B. *Bacteroidaceae* and *Deferribacteraceae*, which varied across sources throughout the experiment. C. *Bifidobacteriaceae*, *Coriobacteriaceae*, *Ruminococcaceae*, and *Verrucomicrobiaceae* were significantly impacted by clindamycin treatment. Examining the relative abundance dynamics throughout the experiment, identified timepoints where relative abundances also significantly varied across sources of mice. Each point represents the relative abundance of an individual mouse stool sample and colored lines represent the mean relative abundances for each source of mice. The gray horizontal line represents the limit of detection. Timepoints where differences across sources of mice were statistically significant by Kruskal-Wallis test with Benjamini-Hochberg correction for testing across multiple days (Table S17) are identified by the asterisk(s) above each timepoint (\*\, P < 0.05, \*\*\, P < 0.01, \*\*\*\, P < 0.001).

\newpage
## Supplementary Tables and Movie
All supplemental material is available at: https://github.com/SchlossLab/Tomkovich_vendor_difs_XXXX_2020/submission. 

**Movie S1. Large shifts in bacterial community structure occurred after clindamycin and *C. difficile* infection.** PCoA of Theta YC distances animated from 0 through 9 days post-infection. PERMANOVA analysis indicated colony source was the variable that explained the most observed variation across fecal communities (source R^2^ = `r rsq_adonis_source`, *P* = `r p_adonis_source`) followed by interactions between cage and day of the experiment. Transparency of the circle corresponds to the day of the experiment, each circle represents a sample from an individual mouse at a specific timepoint. See Table S5 for PERMANOVA results).

**Table S1. *C. difficile* CFU statistical results.**

**Table S2. Mouse weight change statistical results.**

**Table S3. Diversity metrics Kruskal-Wallis statistical results.**

**Table S4. Diversity metrics pairwise Wilcoxon statistical results.**

**Table S5. PERMANOVA results for all mice, all timepoints.**

**Table S6. PERMANOVA results for all mice, all timepoints.**

**Table S7. PERMANOVA results of baseline communities within each source.**

**Table S8. OTUs with relative abudances that significantly vary across sources at baseline, post-clindamycin, or post-infection timepoints.**

**Table S9. Families with relative abudances that significantly vary across sources at baseline, post-clindamycin, or post-infection timepoints.**

**Table S10. OTUs with relative abudances that significantly changed after clindamycin treatment.**

**Table S11. Families with relative abudances that significantly changed after clindamycin treatment.**
**Table S12. Statistical results of logistic regression model performances compared to random chance.**

**Table S13. Pairwise Wilcoxan results of comparing all 6 logistic regression model performances.**

**Table S14. Top 20 most important OTUs for each of the 3 logistic regression models based on OTU relative abundance data.**

**Table S15. Top 20 most important families for each of the 3  logistic regression models based on OTU relative abundance data.**

**Table S16. OTUs with relative abudances that significantly varied across sources of mice on at least 1 day of the experiment.**

**Table S17. Families with relative abudances that significantly varied across sources of mice on at least 1 day of the experiment.**

