---
title: ''
csl: msphere.csl
fontsize: 11pt
output:
  word_document: default
  pdf_document:
    includes:
      in_header: header.tex
    keep_tex: yes
  html_document:
    df_print: paged
geometry: margin=1.0in
bibliography: references.bib
---


```{r knitr_settings, eval=TRUE, echo=FALSE, cache=FALSE, warning=FALSE, message=FALSE, tidy=TRUE}
options(tidyverse.quiet = TRUE)

library(tidyverse)
library(knitr)

opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("message" = FALSE)
opts_chunk$set("cache" = FALSE)

inline_hook <- function(x){
	print(x)

	if(is.list(x)){
		x <- unlist(x)
	}

	if(is.numeric(x)){
		if(abs(x - round(x)) < .Machine$double.eps^0.5){
			paste(format(x,big.mark=',', digits=0, scientific=FALSE))
		} else {
			paste(format(x,big.mark=',', digits=2, nsmall=2, scientific=FALSE))
		}
	} else {
    	paste(x)      
	}
}
knitr::knit_hooks$set(inline=inline_hook)
```



\vspace{35mm}

# The initial gut microbiota and response to antibiotic perturbation influence *Clostridioides difficile* colonization in mice

\vspace{35mm}


Sarah Tomkovich${^1}$, Joshua M.A. Stough${^1}$, Lucas Bishop${^1}$, Patrick D. Schloss^1$\dagger$^

\vspace{40mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

$1$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI 48109




\newpage
\linenumbers


## Abstract
The gut microbiota has a key role in determining susceptibility to *Clostridioides difficile* infections (CDIs). However, much of the mechanistic work examining CDIs in mouse models use animals obtained from a single university colony or vendor. We treated mice from 6 different sources (2 University of Michigan colonies and 4 vendors) with a single clindamycin dose, followed by a *C. difficile* challenge 1 day later and then measured *C. difficile* colonization levels through 9 days post-infection. The microbiota was profiled via 16S rRNA gene sequencing to examine the variation across sources and alterations due to clindamycin treatment and *C. difficile* challenge. While all sources of mice were colonized 1-day post-infection, variation emerged from days 3-7 post-infection with animals from some sources colonized with *C. difficile* for longer and at higher levels. We identified bacteria that varied in relative abundance across sources and throughout the experiment. Some bacteria were consistently impacted by clindamycin treatment in all sources of mice including *Lachnospiraceae*, *Ruminococcaceae*, and *Enterobacteriaceae*. To identify bacteria that were most important to colonization regardless of the source, we created logistic regression models that successfully classified mice based on whether they cleared *C. difficile* by 7 days post-infection using baseline, post-clindamycin, and post-infection community composition data. With these models, we identified 4 bacteria that varied across sources (*Bacteroides*), were altered by clindamycin (*Porphyromonadaceae*), or both (*Enterobacteriaceae* and *Enterococcus*). Microbiota variation across sources better emulates human interindividual variation and can help identify bacterial drivers of phenotypic variation in the context of CDIs.

## Importance
*Clostridioides difficile* is a leading nosocomial infection. Although perturbation to the gut microbiota has been established as a key risk factor, there is variation in who becomes asymptomatically colonized, develops an infection, or has an infection with adverse outcomes. *C. difficile* infection (CDI) mouse models are widely used to answer a variety of *C. difficile* pathogenesis questions. However, the inter-individual variation between mice is less than what is observed in humans, particularly if just one source of mice is used. In this study, we administered clindamycin to mice from 6 different breeding colonies and challenged them with *C. difficile*. Interestingly, only a subset of the bacteria that vary across sources were associated with how long *C. difficile* was able to colonize. Future studies examining the interplay between the microbiota and *C. difficile* should consider using mice from multiple sources to better reflect human interindividual variation.

\newpage

## Introduction

Antibiotics are a common risk factor for *Clostridioides difficile* infections (CDIs), but there is variation in who goes on to develop severe or recurrent CDIs after exposure [@Teng2019; @Kelly2012]. Additionally, asymptomatic colonization, where *C. difficile* is detectable, but symptoms are absent has been documented in infants and adults [@Zacharioudakis2015; @Crobach2018]. The intestinal microbiome has been implicated in asymptomatic colonization [@Zhang2015; @VanInsberghe2020], susceptibility to CDIs [@Mancabelli2017, @Duvallet2017], and adverse CDI outcomes [@Seekatz2016; @Khanna2016; @Pakpour2017; @Lee2020]. 

Mouse models of CDIs have been a great tool for understanding *C. difficile* pathogenesis [@Hutton2014]. The number of CDI mouse model studies has grown substantially since Chen et al. published their C57BL/6 model in 2008, which disrupted the gut microbiota with antibiotics to enable *C. difficile* colonization and symptoms such as diarrhea and weight loss [@Chen2008]. CDI mouse models have been used to examine translationally relevant questions regarding *C. difficile*, including the role of the microbiota and efficacy of potential therapeutics for treating CDIs [@Best2012]. However, variation in the microbiome between mice from the same breeding colony is much less than the variation observed between humans [@Baxter2014; @Nagpal2018]. Additionally, studying the contribution of the microbiota to a particular disease phenotype in one set of lab mice after the same perturbation could yield a number of findings of which only a fraction may be driving the phenotype. 

In the past, our group has attempted to introduce more microbiome variation into the CDI mouse model by using a variety of antibiotic treatments [@Reeves2011; @Schubert2015; @Jenior2017; @Jenior2018]. An alternative approach to maximize microbiome variation is to use mice from multiple sources [@Velazquez2019; @Osbelt2020]. Microbiome differences between different mouse vendors have been well documented and shown to influence susceptibility to a variety of diseases [@Stough2016; @Alegre2019], including enteric infections [@EtienneMesmin2017; @Velazquez2019; @Osbelt2020; @Lai2020; @Thiemann2017; @Rolig2013; @Ge2018]. Additionally, research groups have observed different CDI outcomes in mice despite using similar models and the microbiome has been proposed as one factor potentially mediating CDI susceptibility and outcomes [@Hutton2014; @Lawley2013; @Reeves2011; @Buffie2011; @Jenior2018; @Buffie2014]. Here we examined how variations in the baseline microbiome and responses to clindamycin treatment in C57BL/6 mice from six different sources influenced susceptibility to *C. difficile* colonization and the time needed to clear the infection.

## Results
```{r alpha}
richness_stats_all <- read_tsv("../data/process/richness_stats_all_days.tsv")
padj_dn1_richness <- richness_stats_all %>% filter(day == -1) %>% pull(p.value.adj)

shannon_stats_all <- read_tsv("../data/process/shannon_stats_all_days.tsv")
padj_dn1_shannon <- shannon_stats_all %>% filter(day == -1) %>% pull(p.value.adj)
```

```{r adonis}
adonis_stats_all <- read_tsv("../data/process/adonis_all.tsv")
r_sq_ordered_adonis <- adonis_stats_all %>% arrange(desc(r_sq))
rsq_adonis_source <- r_sq_ordered_adonis %>% pull(r_sq) %>% max
p_adonis_source <- r_sq_ordered_adonis %>% pull(p) %>% max

adonis_stats_dn101 <- read_tsv("../data/process/adonis_dn1-1.tsv")
dn1_rsq_sum <- adonis_stats_dn101 %>% filter(day == -1) %>% select(r_sq) %>% sum
d0_rsq_sum <-  adonis_stats_dn101 %>% filter(day == 0) %>% select(r_sq) %>% sum
d1_rsq_sum <-  adonis_stats_dn101 %>% filter(day == 1) %>% select(r_sq) %>% sum
p_adonis_intial <- adonis_stats_dn101 %>% select(p) %>% max 
p_adonis_min <- adonis_stats_dn101 %>% select(p) %>% min #All p values are same

source_adonis_dn1 <- read_tsv("../data/process/adonis_dn1_source.tsv")
adonis_schloss_rsq_sum <- source_adonis_dn1 %>% filter(source == "Schloss") %>% select(r_sq) %>% sum
p_adonis_schloss_max <- source_adonis_dn1 %>% filter(source == "Schloss") %>% select(p) %>% max
adonis_young_rsq_sum <- source_adonis_dn1 %>% filter(source == "Young") %>% select(r_sq) %>% sum
p_adonis_young_max <- source_adonis_dn1 %>% filter(source == "Young") %>% select(p) %>% max
```

```{r source_taxa}
#Number of different otus across sources at day -1, 0, and 1 timepoints
source_otus <- read_tsv("../data/process/otu_stats_dn1to1_combined.tsv")
source_otus_dn1 <- source_otus %>% filter(day == -1) %>% nrow()
source_otus_d0 <- source_otus %>% filter(day == 0) %>% nrow()
source_otus_d1 <- source_otus %>% filter(day == 1) %>% nrow()

#Function to find which significant otus are shared 
intersect_all <- function(a,b,...){
  Reduce(intersect, list(a,b,...))
}

#OTUs shared across baseline, post-clindamycin, and post-infection timepoints
source_o_dn1 <- source_otus %>% filter(day == -1) %>% pull(otu)
source_o_d0 <- source_otus %>% filter(day == 0) %>% pull(otu)
source_o_d1 <- source_otus %>% filter(day == 1) %>% pull(otu)
shared_source_otus_dn1to1 <- length(intersect_all(source_o_dn1, source_o_d0, source_o_d1))

```

**Microbiota variation is high between mice from different sources.** To test how the microbiotas of mice from different sources impact colonization dynamics after clindamycin exposure, we utilized C57BL/6 mice from 6 different sources: two colonies from the University of Michigan that were split from each other 10 years ago (the Young and Schloss lab colonies) and four commercial source: the Jackson Laboratory, Charles River Laboratories, Taconic Biosciences, and Envigo (which was formerly Harlan). These 4 vendors were chosen because they represent commonly used vendors for CDI studies in mice [@Spinler2016; @Markey2018; @McKee2018; @Yamaguchi2020; @Stroke2018; @EtienneMesmin2017; @Quigley2019; @Mullish2019]. We used 16S rRNA gene sequencing to characterize the fecal bacterial communities across sources at baseline.

Since antibiotics and other risk factors of CDIs are associated with decreased microbiota diversity [@Ross2016], we first examined alpha diversity across the 6 sources of mice. Intially, there was a significant difference in the number of observed operational taxonomic units (OTUs), but not Shannon diversity index (*P*~FDR~ = `r padj_dn1_richness` and *P*~FDR~ = `r padj_dn1_shannon`, respectively) across sources of mice (Fig. 1A-B and Table S1). Next, we compared the community structure of mice from the 6 sources over the course of the experiment using principal coordinate analysis (PCoA) of the $\theta_{YC}$ distances. Permutational multivariate analysis of variance (PERMANOVA) analysis revealed source and cage interactions explained most of the observed variation across fecal communities explained most of the observed community variation (combined R^2^ = `r dn1_rsq_sum`, *P* = `r p_adonis_intial`, Fig. 3 and Table S2). Mice that are co-housed have been shown to have similar gut microbiotas due to coprophagy [@Nguyen2015] and since mice within the same source were housed together, it is not surprising that cage also contributed to the observed microbiota variation.  Since we conducted two experiments approximately 3 months apart, we also compared baseline communities across the 2 experiments. Experiment and cage significantly explained the observed variation only for the Schloss and Young lab mouse colonies (Fig. S2A-B and Table S2). However, most of the vendors also clustered by experiment (Fig. S2C-D, F), suggesting there was some community variation between the 2 experiments within each source, particularly for Schloss, Young, and Envigo mice (Fig. S2G-H). After finding differences at the community level, we next identified the bacteria that initially varied across sources of mice. There were `r source_otus_dn1` OTUs with relative abundances that varied across sources (Fig. 1D and Table S3). By using mice from six sources we were able to increase the number of microbiota communities to test with the clindamycin *C. difficile* colonization mouse model.

```{r cfu}
cfu_stats_all <- read_tsv("../data/process/cfu_stats_all_days.tsv")
padj_d1_cfu <- cfu_stats_all %>% filter(day == 1) %>% pull(p.value.adj) 
median_d1 <- cfu_stats_all %>% filter(day == 1) %>% select(-day, -statistic, -p.value, -parameter, -method, -p.value.adj) %>% pivot_longer(everything(), values_to = "median_cfu", names_to = "Source") %>% arrange(median_cfu)
min_median_d1 <- median_d1 %>%  select(median_cfu) %>% min
max_median_d1 <- median_d1 %>% select(median_cfu) %>% max

cfu_stats_sig <- cfu_stats_all %>% filter(p.value.adj < 0.05)
max_padj_d3to7 <- cfu_stats_sig %>% select(p.value.adj) %>% max

#Percent of mice colonized with C. difficile each day
metadata <- read_tsv("../data/process/processed_metadata.tsv")
#Add clearance_status_dx columns for the rest of the days in addition to d7
metadata_cfu_status <- metadata %>% 
  mutate(clearance_status_d7 = case_when(cfu_d7 > 0 ~ "colonized",
                                         cfu_d7 == 0 ~ "cleared", #changed to match plot labeling used above
                                         cfu_d7 == NA ~ "NA"),
         clearance_status_d1 = case_when(cfu_d1 > 0 ~ "colonized",
                                         cfu_d1 == 0 ~ "cleared", 
                                         cfu_d1 == NA ~ "NA"),
         clearance_status_d2 = case_when(cfu_d2 > 0 ~ "colonized",
                                         cfu_d2 == 0 ~ "cleared", 
                                         cfu_d2 == NA ~ "NA"),
         clearance_status_d3 = case_when(cfu_d3 > 0 ~ "colonized",
                                         cfu_d3 == 0 ~ "cleared", 
                                         cfu_d3 == NA ~ "NA"),
         clearance_status_d4 = case_when(cfu_d4 > 0 ~ "colonized",
                                         cfu_d4 == 0 ~ "cleared", 
                                         cfu_d4 == NA ~ "NA"),
         clearance_status_d5 = case_when(cfu_d5 > 0 ~ "colonized",
                                         cfu_d5 == 0 ~ "cleared", 
                                         cfu_d5 == NA ~ "NA"),
         clearance_status_d6 = case_when(cfu_d6 > 0 ~ "colonized",
                                         cfu_d6 == 0 ~ "cleared", 
                                         cfu_d6 == NA ~ "NA"),
         clearance_status_d8 = case_when(cfu_d8 > 0 ~ "colonized",
                                         cfu_d8 == 0 ~ "cleared", 
                                         cfu_d8 == NA ~ "NA"),
         clearance_status_d9 = case_when(cfu_d9 > 0 ~ "colonized",
                                         cfu_d9 == 0 ~ "cleared", 
                                         cfu_d9 == NA ~ "NA"))


#Percent mice colonized each day, 2 experiments combined----
count_colonized <- function(timepoint, clearance_status_dx){
  metadata_cfu_status %>% 
    filter(day == timepoint, !is.na(cfu)) %>% #get rid of NA values
    count({{ clearance_status_dx }}) %>% 
    mutate(day = timepoint) %>% 
    rename(cfu_status = {{ clearance_status_dx }})
}  

#Figure out number of mice that either cleared C. difficile or were still colonized with C. difficile on each day
day_1 <- count_colonized(1, clearance_status_d1)
day_2 <- count_colonized(2, clearance_status_d2)
day_3 <- count_colonized(3, clearance_status_d3)
day_4 <- count_colonized(4, clearance_status_d4)
day_5 <- count_colonized(5, clearance_status_d5)
day_6 <- count_colonized(6, clearance_status_d6)
day_7 <- count_colonized(7, clearance_status_d7)
day_8 <- count_colonized(8, clearance_status_d8)
day_9 <- count_colonized(9, clearance_status_d9)

#Combine into a data frame and figure out percent colonized per day
percent_colonized_day <- rbind(day_1, day_2, day_3, day_4, day_5, day_6, day_7, day_8, day_9) %>% 
  group_by(day) %>% 
  mutate(total_n = sum(n)) %>% #Figure out total n per day (sum of cleared and colonized counts)
  filter(cfu_status == "colonized") %>% #Only need colonized rows to figure out percent colonized
  mutate(percent_colonized = (n/total_n) * 100)#Add new column for percent_colonized
percent_colonized_d7 <- percent_colonized_day %>% filter(day == 7) %>% 
  pull(percent_colonized)
n_day1 <- percent_colonized_day %>% filter(day == 1) %>% 
  pull(total_n)
n_day2 <- percent_colonized_day %>% filter(day == 2) %>% 
  pull(total_n)
n_day3 <- percent_colonized_day %>% filter(day == 3) %>% 
  pull(total_n)
n_day4 <- percent_colonized_day %>% filter(day == 4) %>% 
  pull(total_n)
n_day5 <- percent_colonized_day %>% filter(day == 5) %>% 
  pull(total_n)
n_day6 <- percent_colonized_day %>% filter(day == 6) %>% 
  pull(total_n)
n_day7 <- percent_colonized_day %>% filter(day == 7) %>% 
  pull(total_n)
n_day8 <- percent_colonized_day %>% filter(day == 8) %>% 
  pull(total_n)
n_day9 <- percent_colonized_day %>% filter(day == 9) %>% 
  pull(total_n)
```
**Clindamycin treatment renders all mice susceptible to *C. difficile* 630 colonization, but clearance time varies across sources.** After the acclimation period, all mice were treated with 10 mg/kg clindamycin via intraperitoneal injection and one day later challenged with 10^3^ *C. difficile* 630 spores (Fig. 2A). Clindamycin was chosen because we have previously demonstrated mice are rendered susceptible, but consistently cleared *C. difficile* within 9 days [@Jenior2018; @Tomkovich2019], clindamycin is frequently implicated with human CDIs [@Guh2018], and is also part of the antibiotic treatment for the frequently cited 2008 CDI mouse model [@Chen2008]. The day after infection, *C. difficile* was detectable in all mice at a similar level (median CFU range: `r format(min_median_d1, digits = 2, scientific=TRUE)`-`r format(max_median_d1, scientific=TRUE, digits = 2)`; *P*~FDR~ = `r padj_d1_cfu`), indicating clindamycin rendered all mice susceptible regardless of source (Fig. 2B). Interestingly, variation in *C. difficile* CFU levels across sources of mice emerged from days 3-7 post-infection (all *P*~FDR~ $\le$ `r max_padj_d3to7`; Fig. 2B and Table S4), suggesting mouse source is associated with *C. difficile* clearance. We conducted two experiments and while the colonization dynamics were similar across most sources of mice, there was some variation between the 2 experiments, particularly for the Schloss and Envigo mice (Fig. S2A-B). Although *C. difficile* 630 causes mild symptoms in mice comparied to other *C. difficile* strains [@Theriot2011], we also saw that weight change significantly varied across sources of mice with the most weight lost two days post-infection (Fig. 2C and Table S5). Importantly, there was also one Jackson and one Envigo mouse that died between 1- and 3-days post-infection during the second experiment. Interestingly, mice obtained from Jackson, Taconic, and Envigo tended to lose more weight, have higher *C. difficile* CFU levels and take longer to clear the infection compared to the other sources of mice (although there was variation between experiments with Schloss and Envigo mice), which was particularly evident 7 days post-infection (Fig. 2B-C, Fig. S2C-D), when `r format(percent_colonized_d7, digits = 1)`% of the mice were still colonized with *C. difficile* (Fig. S2E). By 9 days post-infection the majority of the mice from all sources had cleared *C. difficile* (Fig. 2C) with the exception of 1 Taconic mouse from the first experiment and 2 Envigo mice from the second experiment. Thus, clindamycin rendered all mice susceptible to *C. difficile* 630 colonization, regardless of source, but there was significant variation in disease phenotype across the sources of mice.

```{r clindamycin_taxa}
clindamycin_otus <- read_tsv("../data/process/otu_stats_dn1to0.tsv")
clindamycin_otus_sig <- clindamycin_otus %>% filter(p.value.adj < 0.05) %>% nrow()
```
**Clindamycin treatment alters bacteria in all sources, but a subset of bacterial differences across sources persists.** Given the well known variation in mouse microbiomes across breeding colonies [@Alegre2019], we hypothesized that variation in *C. difficile* clearance would be explained by microbiota variation across the 6 sources of mice. We used 16S rRNA sequencing to characterize microbiota variation across sources after clindamycin treatment and throughout the rest of the experiment. PERMANOVA analysis revealed source was the dominant factor that explained the observed variation across fecal communities (R^2^ = `r rsq_adonis_source`, *P* = `r p_adonis_source`) followed by interactions between cage (mice from the same sources were housed together, primarily at a density of 2 mice per cage) and day of the experiment (Movie S1 and Table S2). Since the majority of the perturbations happened over the initial days of the experiment, we decided to focus on the communities after clindamycin treatment (day 0) and post-infection (day 1).

As expected, clindamycin treatment decreased richness and Shannon diversity across all sources of mice (Fig. 3A-B). Interestingly, significant differences in diversity metrics across sources (*P*~FDR~ < 0.05) emerged after clindamycin treatment, with Charles River mice having higher richness and Shannon diversity than most of the other sources (Fig 3A-B and Table S1). The variation in community structures was also diminished by clindamycin treatment (Fig. 3C and Table S2). Next we identified the bacteria that shifted after clindamycin treatment, regardless of source by analyzing all mice that had sequence data from fecal samples collected at baseline and after clindamycin treatment. We identified `r clindamycin_otus_sig` OTUs that were altered after clindamycin treatment (Fig. 3D and Table S6) in most mice. Despite the significant impact of clindamycin, there were still `r source_otus_d0` OTUs (Fig. 3E and Table S3) with relative abundances that significantly varied across sources. Interestingly, when we compared the list of significant clindamycin impacted bacteria with the bacteria that varied across sources post-clindamycin, we found 4 OTUs (*Enterobacteriaceae (OTU 1)*, *Lachnospiraceae* (OTU 130), *Lactobacillus* (OTU 6), *Enterococcus* (OTU 23)) overlapped (Fig. 3D-E and Tables S3, S6). These findings demonstrate that clindamycin has a consistent impact on the fecal bacterial communities of mice from all sources and only a subset of the OTUs continued to vary across sources.

**Microbiota variation across sources is maintained after *C. difficile* challenge.** Post-infection, significant differences in diversity metrics  were maintained across sources (*P*~FDR~ < 0.05, Fig 4A-B and Table S1). Although the Charles River mice had more diverse microbiotas, the Young and Schloss lab mice were also able to clear *C. difficile* faster than some of the other sources, suggesting microbiota diversity alone does not explain the observed variation in *C. difficile* colonization across sources. Source and cage interaction continued to explain most of the observed community variation (combined R^2^ = `r d1_rsq_sum`, respectively; *P* = `r p_adonis_intial`; Fig. 4D and Table S2). After *C. difficile* challenge, there were `r source_otus_d1` OTUs (Fig. 4D and Table S3) with significantly different relative abundances across sources. In spite of the experimental perturbations that occurred, there were `r shared_source_otus_dn1to1` OTUs with relative abundances that consistently varied across sources at baseline, post-clindamycin, and post-infection. Importantly, some of the OTUs that consistently varied across sources also shifted with clindamycin treatment. For example, *Proteus* increased after clindamycin treatment, but only in Taconic mice. *Enterococcus* was primarily found only in mice purchased from commercial vendors and also increased after clindamycin treatment. In summary, mouse bacterial communities varied significantly according to source throughout the course of the experiment and a consistent subset of bacterial taxa remained different across sources regardless of clindamycin and *C. difficile* challenge.

```{r classification}
models_vs_random <- read_tsv("../data/process/classification_to_random.tsv")
max_p_vs_random <- models_vs_random %>% select(p.value) %>% max
median_otu_day0_AUROC <- models_vs_random %>% filter(model_name == "otu_day0") %>% pull(median)

padj_pairwise_comp <- read_tsv("../data/process/classification_model_pairwise_stats.tsv")
max_padj_otu_day0_comp <- padj_pairwise_comp %>% filter(str_detect(comparison, "otu_day0")) %>% select(p.value.adj) %>% max

#Function to find which significant otus are shared 
intersect_all <- function(a,b,...){
  Reduce(intersect, list(a,b,...))
}

#Read in components of venn diagram script to be able to calculate comparisons:----
#OTUs identified in logistic regression classification models (20 OTUs with the highest ranking for each model)
interp_otus <- read_tsv("../data/process/combined_top20_otus_all_models.tsv")
interp_otus_dn1 <- interp_otus  %>% filter(model_input_day == -1) %>% pull(OTU)
interp_otus_d0 <- interp_otus  %>% filter(model_input_day == 0) %>% pull(OTU)
interp_otus_d1 <- interp_otus  %>% filter(model_input_day == 1) %>% pull(OTU)
interp_combined <- c(interp_otus_dn1, interp_otus_d0, interp_otus_d1)

#OTUs that vary across sources of mice on day -1, 0, or 1
sig_otu_source <- read_tsv("../data/process/otu_stats_dn1to1_combined.tsv")
`sig_otu_day-1` <- sig_otu_source %>% filter(day == -1) %>% pull(otu)
`sig_otu_day0` <- sig_otu_source %>% filter(day == 0) %>% pull(otu)
`sig_otu_day1` <- sig_otu_source %>% filter(day == 1) %>% pull(otu)
#OTUs that consistently vary across sources of mice on day-1, 0, or 1
shared_sig_otus_Dn1toD1 <- intersect_all(`sig_otu_day-1`, sig_otu_day0, sig_otu_day1)


#OTUs that were impacted by clindamycin treatment
`sig_otu_pairs` <- read_tsv("../data/process/otu_stats_dn1to0.tsv") %>% 
  filter(p.value.adj < 0.05) %>% pull(otu)

# Day -1 overlapping OTUs----
# Overlap between OTUs that varied across sources of mice on day -1 and top 20 taxa from logistic regression model based on day -1 community:
dayn1_and_interp_dn1_o <- intersect_all(`sig_otu_day-1`, `interp_otus_dn1`)
# Overlap between OTUs that were altered by clindamycin treatment and top 20 taxa from logistic regression model based on day -1 community:
paired_and_interp_dn1_o <- intersect_all(`sig_otu_pairs`, `interp_otus_dn1`)

# Day 0 overlapping OTUs----
# Overlap between OTUs that varied across sources of mice on day 0 and top 20 taxa from logistic regression model based on day 0 community:
day0_and_interp_d0_o <- intersect_all(`sig_otu_day0`, `interp_otus_d0`)
# Overlap between OTUs that were altered by clindamycin treatment and top 20 taxa from logistic regression model based on day 0 community:
paired_and_interp_d0_o <- intersect_all(`sig_otu_pairs`, `interp_otus_d0`)

# Day 1 overlapping OTUs----
# Overlap between OTUs that varied across sources of mice on day 1 and top 20 taxa from logistic regression model based on day 1 community:
day1_and_interp_d1_o <- intersect_all(`sig_otu_day1`, `interp_otus_d1`)
# Overlap between OTUs that were altered by clindamycin treatment and top 20 taxa from logistic regression model based on day 1 community:
paired_and_interp_d1_o <- intersect_all(`sig_otu_pairs`, `interp_otus_d1`)

#Combined overlapping OTUs based on day -1, 0, and 1 comparisons----
#Combined overlapping OTUs that varied by source and were in the top 20 taxa from at least one logistic regression model & remove any duplicates
dayn1to1_and_interp_combined <- unique(c(`dayn1_and_interp_dn1_o`, `day0_and_interp_d0_o`, `day1_and_interp_d1_o`))
#See which source OTUs show up as significant over multiple days
source_otus_w_duplicates <- c(`dayn1_and_interp_dn1_o`, `day0_and_interp_d0_o`, `day1_and_interp_d1_o`)
key_source_otus <- sort(source_otus_w_duplicates[duplicated(source_otus_w_duplicates)])
#Key source OTUs: "Bacteroides (OTU 2)", "Enterococcus (OTU 23)"
#Combined overlapping OTUs that were altered by clindamycin treatment and were in the top 20 taxa from at least one logistic regression model
paired_and_interp_combined <- unique(c(`paired_and_interp_dn1_o`, `paired_and_interp_d0_o`, `paired_and_interp_d1_o`))
#See which clindamycin-associated OTUs show up as significant over multiple days
clind_otus_w_duplicates <- c(`paired_and_interp_dn1_o`, `paired_and_interp_d0_o`, `paired_and_interp_d1_o`)
key_clind_otus <- sort(clind_otus_w_duplicates[duplicated(clind_otus_w_duplicates)])
#List of key OTUs from source and clindamycin varying OTUs:
key_otus <- unique(c(key_source_otus, key_clind_otus))
list_key_otus <- paste(key_otus, collapse= ", ") #Format to output in paper

#Venn diagram calculations for OTUs----
venn_otus_source <- length(dayn1to1_and_interp_combined)
venn_otus_intersect <- length(intersect(dayn1to1_and_interp_combined, paired_and_interp_combined))
venn_otus_clind <-  length(paired_and_interp_combined)
```
**Baseline, post-clindamycin, and post-infection community data can predict mice that will clear *C. difficile* by 7 days post-infection.** After identifying taxa that varied by source, changed after clindamycin treatment, or both, we next wanted to determine which taxa were influencing the variation in *C. difficile* colonization at day 7 (Fig. 2D, Fig. S2C). We trained three L2-regularized logistic regression models with input bacterial community data from the baseline, post-clindamycin, and post-infection timepoints of the experiment to predict *C. difficile* colonization status on day 7 (Fig. S3A-B). All models were better at predicting *C. difficile* colonization status on day 7 than random chance (all *P* $\le$ `r format(max_p_vs_random, scientific = TRUE, digits = 2)`; Table S7). Interestingly, the model based on the post-clindamycin (day 0) community OTU data performed significantly better than the other models with an area under the receiving operator characteristic curve (AUROC) of `r median_otu_day0_AUROC` (*P*~FDR~ $\le$ `r format(max_padj_otu_day0_comp, scientific = TRUE, digits = 2)` for pairwise comparisons; Table S7). Thus, we were able to use bacterial relative abundance data alone to differentiate mice that had cleared *C. difficile* before day 7 from the mice still colonized with *C. difficile* at that timepoint. Interestingly, the model built with OTU relative abundance data post-clindamycin treatment had the best performance, suggesting how the bacterial community responds to clindamycin treatment has the greatest influence on subsequent *C. difficile* colonization dynamics.

Next, to examine the bacteria that were driving each model’s performance, we pulled out the top 20 OTUs that had the highest absolute feature weights in each of the 6 models (Table S8). First, we looked at OTUs from the model with the best performance that was based on the post-clindamycin treatment bacterial community data. While most of the 20 OTUs had low relative abundances on day 0, *Enterobacteriace*, *Bacteroides* and *Proteus* had high relative abundances in at least one source of mice and significantly varied across sources (Fig. 5A). Next, the top 20 taxa from each model were compared to the list of taxa that varied across source at the same timepoint (Fig. 1D, 3E, 4D and Table S3) and the taxa that were altered by clindamycin treatment (Fig. 3D and Table S6). We found a subset of OTUs that were important to the model and overlapped with bacteria that varied by either source, clindamycin treatment, or both (Fig. S4). Combining the overall results for the 3 models identified `r venn_otus_source` OTUs associated with source, `r venn_otus_clind` OTUs associated with clindamycin treatment, and `r venn_otus_intersect` OTUs associated with both (Fig. 5B). Several OTUs (*`r list_key_otus`*) appeared across at least 2 models, so we examined how the relative abundances of these OTUs varied over the course of the experiment (Fig. 6). Throughout the experiment, there was at least 1 timepoint where relative abundances of these OTUs significantly varied across sources (Table S9). Interestingly, there were no OTUs that emerged as consistently enriched or depleted in mice that were colonized past 7 days post-infection with *C. difficile* 630, suggesting multiple bacteria influence the time needed to clear the infection. Together, these results suggest the initial bacterial communities and their responses to clindamycin influence the clearance of *C. difficile*.

## Discussion

By running our CDI model with mice from 6 different sources, we were able to identify bacterial taxa that were unique to sources throughout the experiment as well as taxa that were universally impacted by clindamycin. We trained L2 logistic regression models with baseline, post-clindamycin treatment, and post-infection fecal community data that could predict whether mice cleared *C. difficile* by 7 days post-infection better than random chance. We identified *`r list_key_otus`* (Fig. 6) as candidate bacteria within these communities that were influencing variation in *C. difficile* colonization dynamics since these bacteria were all important in the logistic regression models and varied by source, were impacted by clindamycin treatment, or both. Overall, our results demonstrate clindamycin is sufficient to render mice from multiple sources susceptible to CDI and only a subset of the interindividual microbiota variation across mice from different sources was associated with the time needed to clear *C. difficile*. 

Other studies have taken similar approaches by using mice from multiple sources to identify bacteria that either promote colonization resistance or increase susceptibility to enteric infections [@Velazquez2019; @EtienneMesmin2017; @Osbelt2020; @Lai2020; @Thiemann2017; @Rolig2013; @Ge2018]. For example, in the context of *Salmonella* infections, *Enterobacteriaceae* and segmented filamentous bacteria have emerged as protective [@Velazquez2019; @Lai2020]. A previous study with *C. difficile* identified an endogenous protective *C. difficile* strain LEM1 that bloomed after antibiotic treatment in mice from Jackson or Charles River Laboratories, but not Taconic that protected mice against the more toxigenic *C. difficile* VPI10463 [@EtienneMesmin2017]. Given that we obtained mice from the same vendors, we checked all mice for endogenous *C. difficile* by plating stool samples that were collected after clindamycin treatment. However, we did not identify any endogenous *C. difficile* strains prior to challenge, suggesting there were no endogenous protective strains in the mice we received and other bacterial taxa mediated the variation in *C. difficile* colonization across sources. 

Although all mice were susceptible to *C. difficile* colonization, mice from Jackson, Taconic, and Envigo mice tended to remain colonized through at least 7 days post-infection. We identified a subset of bacteria that were important in predicting whether a mouse was still colonized with *C. difficile* 7 days post-infection. These results suggest a subset of the bacterial community is responsible for determining the length of time needed to clear *C. difficile* colonization. 

Differences in CDI mouse model studies have been attributed to different intestinal microbiotas of mice from different sources. For example, groups using the same clindamycin treatment and C57BL/6 mice had different *C. difficile* outcomes, one having sustained colonization [@Buffie2011], while the other had transient [@Reeves2011], despite both using *C. difficile* VPI 10643. Baseline differences in the microbiota composition have been hypothesized to partially explain the differences in colonization outcomes and overall susceptibility to *C. difficile* after treatment with the same antibiotic [@Lawley2013; @Hutton2014]. The bacterial perturbations induced by clindamycin treatment have been well characterized and our findings agree with previous CDI mouse model work demonstrating *Enterococcus* and *Enterobacteriaceae* were associated with *C. difficile* susceptibility and *Porpyhromonadaceae*, *Lachnospiraceae*, *Ruminococcaceae*, and *Turicibacter* were associated with resistance [@Schubert2015; @Jenior2018; @Tomkovich2019; @Buffie2011; @Buffie2014; @Lawley2009; @Lawley2012; @Jump2014]. While we have demonstrated that susceptibility is uniform across sources of mice after clindamycin treatment, there could be different outcomes for either susceptibility or clearance in the case of other antibiotic treatments. The *C. difficile* strain used could also be contributing to the variation in *C. difficile* outcomes seen across different research groups. For example, a group found differential colonization outcomes after clindamycin treatment, with *C. difficile* 630 and M68 eventually becoming undectable while BI-7 remained detectable up to 70 days post-treatment [@Lawley2012]. We found the time needed to naturally clear *C. difficile* varied across sources of mice implying that at least in the context of the same perturbation, microbiota differences seemed to influence infection outcome more than susceptibility. More importantly, we were able to reduce the variation observed across sources to identify a subset of OTUs that were also important for predicting *C. difficile* colonization status 7 days post-infection. Since all but 3 mice eventually cleared *C. difficile* 630 by 9 days post-infection and the model built with the post-clindamycin OTU relative abundance data had the best performance, our results suggest clindamycin treatment had a large role in determining *C. difficile* susceptibility and clearance than the source of the mice.

Our approach successfully increased the diversity of murine bacterial communities tested in our clindamycin *C. difficile* model. One alternative approach that has been used in some CDI studies [@NagaoKitamoto2020; @Battaglioli2018; @Robinson2014; @Collins2015; @Collins2018; @Hryckowian2018] is to associate mice with human microbiotas. However, a major caveat to this method is the substantial loss of human microbiota community members upon transfer to mice [@Fouladi2020; @Walter2020]. Additionally with the exception of 2 recent studies [@NagaoKitamoto2020; @Battaglioli2018], most of the CDI mouse model studies to date associated mice with just 1 types of human microbiota either from a single donor or a single pool from multiple donors [@Robinson2014; @Collins2015; @Collins2018; @Hryckowian2018], which does not aid in the goal of modeling the interpersonal variation seen in humans to understand how the microbiota influences susceptiblity to CDIs and adverse outcomes. Importantly, our study using mice from 6 different sources increased the variation between groups of mice compared to using 1 source alone, to better reflect the inter-individual microbiota variation observed in humans. Encouragingly, decreased *Bifidobacterium*, *Porphyromonas*, *Ruminococcaceae* and *Lachnospiraceae* and increased *Enterobacteriaceae*, *Enterococcus*, *Lactobacillus*, and *Proteus* have all been associated with human CDIs [@Mancabelli2017, @Duvallet2017] and were well represented in our study, suggesting most of the mouse sources are suitable for gaining insights into microbiota associated factors influencing *C. difficile* colonization and infections in humans. An important exception was *Enterococcus*, which was primarily absent from the mice from University of Michigan colonies and *Proteus*, which was only found in Taconic mice. Importantly, the fact that some CDI-associated bacteria were only found in a subset of mice has important implications for future CDI mouse model studies. 

There are several limitations to our work. The microbiome is composed of viruses, fungi, and parasites in addition to bacteria, and these non-bacterial members can also vary across mouse sources [@Rasmussen2019; @Mims2020]. While our study focused solely on the bacterial portion, viruses and fungi have also begun to be implicated in the context of CDIs or FMT treatments for recurrent CDIs [@Markey2018; @Stewart2019; @Ott2017; @Zuo2017; @Zuo2018]. Beyond community composition, the metabolic function of the microbiota also has a CDI signature [@Robinson2019; @Jenior2017; @Fletcher2018; @Jump2014] and can vary across mice from different sources [@Xiao2015]. For example, microbial metabolites, particularly secondary bile acids and butyrate production, have been implicated as important contributors to *C. difficile* resistance [@Buffie2014; @Lawley2012]. Although, we only looked at composition, *Ruminococcaceae* and *Lachnospiraceae* both emerged as important taxa for classifying day 7 *C. difficile* colonization status and metagenomes from these bacteria have been shown to contain the bile acid-inducible gene cluster necessary for secondary bile acid formation and ability to produce butyrate [@Vital2019; @Collins2015]. Interestingly, butyrate has previously been shown to vary across vendors and mediated resistance to *Citrobacter rodentium* infection, a model of enterohemorrhagic and enteropathogenic *Escherichia coli* infections [@Osbelt2020]. Evidence for immunological toning differences in IgA and Th17 cells across mice from different vendors have also been documented and [@Fransen2015; @Ivanov2009] may also influence response to CDI, particularly in the context of severe CDIs [@Azrad2018; @Saleh2019]. The outcome after *C. difficile* exposure depends on a multitude of factors, including age, diet, and immunity; all of which are also influenced by the microbiota. 

We have demonstrated that the ways baseline microbiotas from different mouse sources respond to clindamycin treatment influences the length of time mice remained colonized with *C. difficile* 630. For those interested in dissecting the contribution of the microbiome to *C. difficile* pathogenesis and treatments, using multiple sources of mice may yield more insights than a single model alone. Furthermore, for studies wanting to examine the interplay between a particular bacterial taxon such as *Enterococcus* and *C. difficile*, these results could serve as a resource for selecting which mice to order to address the question. Using mice from multiple sources helps model the interpersonal microbiota variation among humans to aid our understanding of how the gut microbiota contributes to CDIs.

\newpage

## Acknowledgements
This work was supported by the National Institutes of Health (U01AI124255). ST was supported by the Michigan Institute for Clincial and Health Research Postdoctoral Translation Scholars Program (UL1TR002240). We thank members of the Schloss lab for feedback on planning the experiments and data presentation, as well as code tutorials and feedback through Code club. In particular, we want to thank Begüm Topçuoğlu for help with implementing L2 logistic regression models using her [pipeline](https://github.com/SchlossLab/ML_pipeline_microbiome), Ana Taylor for help with media preparation and sample collection, and Nicholas Lesniak for his critical feedback on the manuscript. We also thank members of Vincent Young’s lab, particularly  Kimberly Vendrov, for guidance with the *C. difficile* infection mouse model and donating the mice. We also want to thank the Unit for Laboratory Animal Medicine at the University of Michigan for maintaining our mouse colony and providing the institutional support for our mouse experiments. Finally, we thank Kwi Kim, Austin Campbell, and Kimberly Vendrov for their help in maintaining the Schloss lab's anaerobic chamber.

\newpage

## Materials and Methods

**(i) Animals.** All experiments were approved by the University of Michigan Animal Care and Use Committee (IACUC) under protocol number PRO00006983. Female C57BL/7 mice were obtained from 6 different sources: The Jackson Laboratory, Charles River Laboratories, Taconic Biosciences, Envigo, and two colonies at the University of Michigan (the Schloss lab colony and the Young lab colony). The Young lab colony was originally established with mice purchased from Jackson, and the Schloss lab colony was established 10 years ago with mice donated from the Young lab. The 4 groups of mice purchased from vendors were allowed to acclimate to the University of Michigan mouse facility for 13 days prior to starting the experiment. At least 4 female mice (age 5-10 weeks) were obtained per source and mice from the same source were primarily housed at a density of 2 mice per cage. The experiment was repeated once, approximately 3 months after the start of the first experiment.

**(ii) Antibiotic treatment.** After the 13-day acclimation period and 1 day prior to challenge (Fig. 1A), all mice received 10 mg/kg clindamycin (filter sterilized through a 0.22 micron syringe filter prior to administration) via intraperitoneal injection.

**(iii) *C. difficile* infection model.** Mice were challenged with 10^3^ spores of *C. difficile* strain 630 via oral gavage post-infection 1 day after clindamycin treatment as described previously [@Jenior2018]. Mice weights and stool samples were taken daily through 9 days post-challenge. Collected stool was split for *C. difficile* CFU quantification and 16S rRNA sequencing analysis. *C. difficile* quantification stool samples were transferred to the anaerobic chamber, serially diluted in PBS, plated on taurocholate-cycloserine-cefoxitin-fructose agar (TCCFA) plates, and counted after 24 hours of incubation at 37&deg;C under anaerobic conditions. A sample from the day 0 timepoint (post-clindamycin and prior to *C. difficile* challenge) was also plated on TCCFA to ensure mice were not already colonized with *C. difficile* prior to infection. There were 3 deaths recorded over the course of the experiment, 1 Taconic mouse died prior to *C. difficile* challenge and 1 Jackson and 1 Envigo mouse died between 1- and 3-days post-infection. Mice were categorized as cleared when no *C. difficile* was detected in the first serial dilution (limit of detection: 100 CFU). Stool samples for 16S rRNA sequencing were snap frozen in liquid nitrogen and stored at -80&deg;C until DNA extraction.

**(iv) 16S rRNA sequencing.** DNA was extracted from -80&deg;C stored stool samples using the DNeasy Powersoil HTP 96 kit (Qiagen) and an EpMotion 5075 automated pipetting system (Eppendorf). The V4 region was amplified for 16S rRNA with the AccuPrime Pfx DNA polymerase (Thermo Fisher Scientific) using custom barcoded primers, as previously described [@Kozich2013]. The ZymoBIOMICS microbial community DNA standards was used as a mock community control [@Sze2019] and water was used as a negative control per 96-well extraction plate. The PCR amplicons were cleaned up and normalized with the SequalPrep normalization plate kit (Thermo Fisher Scientific). Amplicons were pooled and quantified with the KAPA library quantification kit (KAPA biosystems), prior to sequencing using the MiSeq system (Illumina).

**(v) 16S rRNA gene sequence analysis.** mothur (v. 1.43) was used to process all sequences [@Schloss2009] with a previously published protocol [@Kozich2013]. Reads were combined and aligned with the SILVA reference database [@Quast2012]. Chimeras were removed with the VSEARCH algorithm and taxonomic assignment was completed with a modified version (v16) of the Ribosomal Database Project reference database (v11.5) [@Cole2013] with an 80% cutoff. Operational taxonomic units (OTUs) were assigned with a 97% similarity threshold using the opticlust algorithm [@Westcott2017]. To account for uneven sequencing across samples, samples were rarefied to 5,437 sequences 1,000 times for alpha and beta diversity analyses. PCoAs were generated based on $\theta_{YC}$ distances. Permutational multivariate analysis of variance (PERMANOVA) was performed on mothur-generated $\theta_{YC}$ distance matrices with the adonis function in the vegan package [@Vegan2018] in R [@r_citation_2018].

**(vi) Classification model training and evaluation.** Models were generated based on mice that were categorized as either cleared or colonized 7 days post-infection and had sequencing data from the baseline (day -1), post-clindamycin (day 0), and post-infection (day 1) timepoints of the experiment. Input bacterial community relative abundance data at the OTU level from the baseline, post-clindamycin, and post-infection timepoints was used to generate 6 classification models that predicted *C. difficile* colonization status 7 days post-infection. The L2-regularized logistic regression models were trained and tested using the caret package [@Kuhn2008] in R as previously described [@Topcuoglu2020] with the exception that we used 60% training and 40% testing data splits for the cross-validation of the training data to select the best cost hyperparameter and the testing of the held out test data to measure model performance. The modified training to testing ratio was selected to accommodate the small number of samples in the dataset. Code was modified from https://github.com/SchlossLab/ML_pipeline_microbiome to update the classification outcomes and change the data split ratios. The modified repository to regenerate this analysis is available at https://github.com/tomkoset/ML_pipeline_microbiome.

**(vii) Statistical analysis.** All statistical tests were performed in R (v `r paste(version$major, ".", version$minor, sep = "")`) [@r_citation_2018]. The Kruskal-Wallis test was used to analyze differences in *C. difficile* CFU, mouse weight change, and alpha diversity across sources with a Benjamini-Hochberg correction for testing multiple timepoints, followed by pairwise Wilcoxon comparisons with Benjamini-Hochberg correction. For taxonomic analysis and generation of logistic regression model input data, *C. difficile* (OTU 20) was removed. Bacterial relative abundances that varied across sources at the OTU level were identified with the Kruskal-Wallis test with Benjamini-Hochberg correction for testing all identified OTUs, followed by pairwise Wilcoxon comparisons with Benjamini-Hochberg correction. OTUs impacted by clindamycin treatment were identified using the Wilcoxon signed rank test with matched pairs of mice samples for day -1 and day 0. To determine whether classification models had better performance (test AUROCs) than random chance (0.5), we used the one-sample Wilcoxon signed rank test. To examine whether there was an overall difference in predictive performance across the 6 classification models we used the Kruskal-Wallis test followed by pairwise Wilcoxan comparisons with Benjamini-Hochberg correction for multiple hypothesis testing. The tidyverse package was used to wrangle and graph data (v `r paste(packageVersion("tidyverse")[1])`) [@Tidyverse2019].

**(viii) Code availability.** Code for all data analysis and generating this manuscript is available at https://github.com/SchlossLab/Tomkovich_Vendor_XXXX_2020.

**(ix) Data availability.** The 16S rRNA sequencing data have been deposited in the National Center for Biotechnology Information Sequence Read Archive (BioProject Accession no. PRJNA608529).

\newpage

## References

<div id="refs"></div>

\newpage

## Figures
```{r sample_size}
#figure out in functions.R and export results as a tsv to read in
analysis_n <- read_tsv("../data/process/processed_metadata.tsv")

#Initial N for each source from the 2 experiments combined:
schloss_i <- analysis_n %>% filter(day == -1 & vendor == "Schloss") %>% nrow()
young_i <- analysis_n %>% filter(day == -1 & vendor == "Young") %>% nrow()
jackson_i <- analysis_n %>% filter(day == -1 & vendor == "Jackson") %>% nrow()
cr_i <- analysis_n %>% filter(day == -1 & vendor == "Charles River") %>% nrow()
taconic_i <- analysis_n %>% filter(day == -1 & vendor == "Taconic") %>% nrow()
envigo_i <- analysis_n %>% filter(day == -1 & vendor == "Envigo") %>% nrow()

#Range in N for C. difficile CFU quantification:
cfu_n_by_day <- analysis_n %>% 
  filter(!is.na(cfu)) %>% #remove rows with NAs
  group_by(day) %>% count() %>% arrange(n) %>% ungroup()
min_cfu_n <- cfu_n_by_day %>% select(n) %>% min
max_cfu_n <- cfu_n_by_day %>% select(n) %>% max

#Range in N for mouse weight change data over the course of the experiment:
weight_n_by_day <- analysis_n %>% 
  filter(!is.na(weight)) %>% #remove rows with NAs
  group_by(day) %>% count() %>% arrange(n) %>% ungroup()
min_weight_n <- weight_n_by_day %>% select(n) %>% min
max_weight_n <- weight_n_by_day %>% select(n) %>% max

#C. difficile day 7 colonization status N range across sources:
day7_status_n <- analysis_n %>% 
  filter(!is.na(clearance_status_d7)) %>% #remove rows with NAs
  filter(day == 7) %>% 
  group_by(vendor) %>% count() %>% arrange(n) %>% ungroup
day7_status_n_min <- day7_status_n %>% select(n) %>% min
day7_status_n_max <-day7_status_n %>% select(n) %>% max

#N for mice with sequencing data on both day -1 and day 0 (used to determine taxa that were altered by clindamycin treatment)
n_paired_for_clind <- read_tsv("../data/process/vendors.diversity.summary") %>% #read in sequencing data & join to analysis_n
  filter(method == "ave") %>% 
  select(group, sobs, shannon, invsimpson, coverage) %>% 
  inner_join(analysis_n, by = c("group" = "id")) %>% 
  filter(day == -1 | day == 0) %>% 
  filter(duplicated(mouse_id)) %>% 
  nrow()
```
\includegraphics{figure_1.pdf}
**Figure 1. Microbiota variation is high between mice from different sources.** A-B. Number of observed OTUs (A) and Shannon diversity index values (B) across sources of mice at baseline (day -1) of the experiment. Differences across sources were analyzed by Kruskal-Wallis test with Benjamini-Hochberg correction for testing each day of the experiment and the adjusted *P* value was < 0.05 for panel A (Table S1). None of the *P* values from pairwise Wilcoxon comparisons between sources were significant after Benjamini-Hochberg correction (Table S1). Gray lines represent the median values for each source of mice. C. Principal Coordinates Analysis (PCoA) of $\theta_{YC}$ distances of baseline stool samples. Permutational multivariate analysis of variance (PERMANOVA) analysis demonstrated that source and the interaction between source and cage explained most of the variation (combined R^2^ = `r dn1_rsq_sum`, *P* = `r p_adonis_intial`,  see Table S2). For A-C:  each symbol representing the value for a stool sample from an individual mouse, circles represent experiment 1 mice and triangles represent experiment 2 mice. D. Plots highlighting the median (point) and interquantile range (colored lines) of the relative abundances for the top 20 bacteria out of the `r source_otus_dn1` OTUs that varied across sources at baseline (Table S3).

\newpage
\includegraphics{figure_2.pdf}

**Figure 2. Clindamycin is sufficient to promote *C. difficile* colonization in all mice, but clearance time varies across sources of C57BL/6 mice.** A. Setup of the experimental timeline. Mice for the experiments were obtained from 6 different sources: the Schloss (N = `r schloss_i`) and Young lab (N = `r young_i`) colonies at the University of Michigan, the Jackson Laboratory (N = `r jackson_i`), Charles River Laboratory (N = `r cr_i`), Taconic Biosciences (N = `r taconic_i`), and Envigo (N = `r envigo_i`). All mice were administered 10 mg/kg clindamycin intraperitoneally (IP) 1 day before challenge with *C. difficile* 630 spores on day 0. Mice were weighed and feces was collected daily through the end of the experiment (9 days post-infection). Note: 3 mice died during course of experiment. 1 Taconic mouse prior to infection and 1 Jackson and 1 Envigo mouse between 1- and 3-days post-infection. B. *C. difficile* CFU/gram stool measured over time (N = `r min_cfu_n`-`r max_cfu_n` mice per timepoint) via serial dilutions. The black line represents the limit of detection for the first serial dilution. CFU quantification data was not available for each mouse due to early deaths, stool sampling difficulties, and not plating all of the serial dilutions. C. Mouse weight change measured in grams over time (N = `r min_weight_n`-`r max_weight_n` mice per timepoint), all mice were normalized to the weight recorded 1 day before infection. For B-C: timepoints where differences across sources of mice were statistically significant by Kruskal-Wallis test with Benjamini-Hochberg correction for testing across multiple days (Table S4 and Table S5) are reflected by the asterisk(s) above each timepoint (\*\, *P* < 0.05). Lines represent the median for each source and circles represent individual mice from experiment 1 while triangles represent mice from experiment 2. 

\newpage
\includegraphics{figure_3.pdf}
**Figure 3. Clindamycin treatment alters bacteria in all sources, but a subset of bacterial differences across sources persists.** A-B. Number of observed OTUs (A) and Shannon diversity index values (B) across sources of mice after clindamycin treatment (day 0). Data were analyzed by Kruskal-Wallis test with Benjamini-Hochberg correction for testing each day of the experiment and the adjusted *P* value was < 0.05. Significant *P* values from the pairwise Wilcoxon comparisons between sources with Benjamini-Hochberg correction are shown (Table S1). C. PCoA of $\theta_{YC}$ distances from stools collected post-clindamycin. PERMANOVA analysis demonstrated that source and the interaction between source and cage explained most of the variation observed post-clindamycin (combined R^2^ = `r d0_rsq_sum`, *P* = `r p_adonis_intial`, see Table S2).For A-C, each symbol represents a stool sample from an individual mouse, with circles representing experiment 1 mice and triangles representing experiment 2 mice. D. Plots highlighting the median (point) and interquantile range (colored lines) of the top 10 OTUs out of `r clindamycin_otus_sig` with relative abundances that changed after clindamycin treatment (adjusted *P* value < 0.05). Data were analyzed by Wilcoxon signed rank test limited to mice that had paired sequence data for day -1 and 0 (N = `r n_paired_for_clind`), with Benjamini-Hochberg correction for testing all identified OTUs (Table S6). The gray vertical line indicates the limit of detection. E. Plots highlighting the median (point) and interquantile range (colored lines) of the relative abundances for the `r source_otus_d0` OTUs that varied across sources after clindamycin treatment (Table S3).

\newpage
\includegraphics{figure_4.pdf}
**Figure 4. Microbiota variation across sources is maintained after *C. difficile* challenge.** A-B. Number of observed OTUs (A) and Shannon diversity index values (B) across sources of mice post-infection (day 1). Data were analyzed by Kruskal-Wallis test with Benjamini-Hochberg correction for testing each day of the experiment and the adjusted *P* value was < 0.05. Significant *P* values from the pairwise Wilcoxon comparisons between sources with Benjamini-Hochberg correction are shown (Table S1). PCoA of $\theta_{YC}$ distances of post-infection stool samples. PERMANOVA analysis demonstrated that source and the interaction between source and cage explained most of the variation (combined R^2^ = `r d1_rsq_sum`, *P* = `r p_adonis_intial`, Table S2). For A-C:  each symbol representing the value for a stool sample from an individual mouse, circles represent experiment 1 mice and triangles represent experiment 2 mice. D. Plots highlighting the median (point) and interquantile range (colored lines) of the relative abundances for the top 20 bacteria out of the `r source_otus_d1` OTUs that varied across sources post-infection. The gray vertical line indicates the limit of detection. For each timepoint OTUs with differential relative abundances across sources of mice were identified by Kruskal-Wallis test with Benjamini-Hochberg correction for testing all identified OTUs (Table S3).

\newpage
\includegraphics{figure_5.pdf}
**Figure 5. Bacteria that influence whether mice cleared *C. difficile* by day 7.** A. Baseline relative abundance data for 3 of the OTUs from the classification model based on day 0 OTU relative abundances that significantly varied across sources of mice and had high relative abundances in the community. Symbols represent the relative abundance data for an individual mouse, circles represent mice that cleared *C. difficile* by day 7, X-shapes represent mice that were still colonized with *C. difficile*, and open circles represent mice that did not have *C. difficile* CFU counts for day 7 post-infection. Gray lines indicate the median relative abundances for each source. Asterisks are shown for pairwise Wilcoxon comparisons with Benjamini-Hochberg correction where *P* < 0.05. B. Venn diagram that combines Fig. S4 summaries of OTUs that were important to the day -1, 0, and 1 classification models (Table S8) and either overlapped with taxa that varied across sources at the same timepoint, were impacted by clindamycin treatment, or both. See Fig. S4 for separate comparisons of taxa from the day -1, 0, and 1 classification models. Red OTUs were important to more than 1 classification model.

\newpage
\includegraphics{figure_6.pdf}
**Figure 6: Key OTUs vary across sources throughout the experiment.** A-D. Relative abundances of red OTUs from Fig. 5A  that were important for at least two classification models are shown over time. A. *Bacteroides* (OTU 2), which varied across sources throughout the experiment. B-C. *Enterobacteriaceae* (B) and *Enterococcus* (C), which significantly varied across sources and were impacted by clindamycin treatment. D. *Porphyromonadaceae* (OTU 7), which was significantly impacted by clindamycin treatment and examining relative abundance dynamics over the course of the experiment, revealed timepoints where relative abundances also significantly varied across sources of mice. Symbols represent the relative abundance data for an individual mouse, circles represent mice that cleared *C. difficile* by day 7, X-shapes represent mice that were still colonized with *C. difficile*, and open circles represent mice that did not have *C. difficile* CFU counts for day 7 post-infection. Colored lines indicate the median relative abundances for each source. The gray horizontal line represents the limit of detection. Timepoints where differences across sources of mice were statistically significant by Kruskal-Wallis test with Benjamini-Hochberg correction for testing across multiple days (Table S9) are identified by the asterisk(s) above each timepoint (\*\, P < 0.05).

\newpage
\includegraphics{figure_S1.pdf}
**Figure S1. Bacterial communities vary between experiments for some sources.** A-F. PCoA of $\theta_{YC}$ distances for the baseline fecal bacterial communities within each source of mice. Each symbol represents a stool sample from an individual mouse with color corresponding to experiment and shape representing cage mates. PERMANOVA was performed within each group to examine the contributions of experiment and cage to observed variation. Experiment number and cage only significantly explained observed variation for mice from the Schloss (combined R^2^ = `r adonis_schloss_rsq_sum`; *P* $\le$ `r p_adonis_schloss_max`) and Young (combined R^2^ = `r adonis_young_rsq_sum`; *P* $\le$ `r p_adonis_young_max`) lab colonies (Table S2). G-H: Boxplots of the $\theta_{YC}$ distances of the 6 sources of mice relative to mice within the same source and experiment (G) or mice within the same source and between experiments (H) at baseline. Symbols represent individual mouse samples: circles for experiment 1 and triangles for experiment 2. 

\newpage
\includegraphics{figure_S2.pdf}
**Figure S2. *C. difficile* CFU variation across sources varies slightly between the 2 experiments.** A-B. *C. difficile* CFU/gram of stool quantification over time for experiment 1 (A) and 2 (B). Experiments were conducted approximately 3 months apart. Lines represent the median CFU for each source, symbols represent individual mice and the black line represents the limit of detection. C. *C. difficile* CFU/gram stool on day 7 post-infection across sources of mice with asterisks for pairwise Wilcoxon comparisons with Benjamini-Hochberg correction where *P* < 0.05. D. Mouse weight change 2 days post-infection across sources of mice, no pairwise Wilcoxon comparisons were significant after Benjamini-Hochberg correction. For C-D: circles represent experiment 1 mice, triangles represent experiment 2 mice and gray lines indicate the median values for each group. E. Percent of mice that were colonized with *C. difficile* over the course of the experiment. Each day the percent is calculated based on the mice where *C. difficile* CFU was quantified for that particular day. Total N for each day: day 1 (N = `r n_day1`), day 2 (N = `r n_day2`), day 3 (N = `r n_day3`), day 4 (N = `r n_day4`), day 5 (N = `r n_day5`), day 6 (N = `r n_day6`), day 7 (N = `r n_day7`), day 8 (N = `r n_day8`), and day 9 (N = `r n_day9`).

\newpage
\includegraphics{figure_S3.pdf}
**Figure S3. Bacterial community composition before, after clindamycin perturbation, and post-infection can predict *C. difficile* colonization status 7 days post-challenge.** A. Bar graph visualizations of overall day 7 C. difficile colonization status that were used as classification outcomes to build L2-regularized logistic regression models. Mice were classified as colonized or cleared (not detectable at the limit of detection of 100 CFU) based on CFU g/stool data from 7 days post-infection. B. *C. difficile* CFU status on Day 7 within each mouse source. N = `r day7_status_n_min`-`r day7_status_n_max` mice per group. C. L2-regularized logistic regression classification model area under the receiving operator characteristic curve (AUROCs) to predict *C. difficile* CFU on day 7 post-infectoin (Fig. 1D, Fig. S3) based on the OTU community relative abundances at baseline (day -1), post-clindamycin (day 0), and post-infection (day 1). All models performed better than random chance (AUROC = 0.5; all *P* $\le$ `r format(max_p_vs_random, scientific = TRUE, digits = 2)`; Table S7) and the model built with post-clindamycin treated bacterial OTU relative abundances had the best performance ((*P*~FDR~ $\le$ `r format(max_padj_otu_day0_comp, scientific = TRUE, digits = 2)` for all pairwise comparisons; Table S7). For list of the 20 OTUs that were ranked as most important to each model, see Table S8.

\newpage
\includegraphics{figure_S4.pdf}
**Figure S4. OTUs from classification models based on baseline, post-clindamycin treatment, or post-infection community data vary by source, clindamycin treatment, or both.** A-C. Venn diagrams of top 20 important OTUs from baseline (A), post-clindamycin treatment (B), and post-infection (C) classification models (Table S8) that overlapped with OTUs that varied across sources at the corresponding timepoint, were impacted by clindamycin treatment, or both. Bold OTUs signify OTUs that were important to more than 1 classification model.

\newpage
## Supplementary Tables and Movie
All supplemental material is available at: https://github.com/SchlossLab/Tomkovich_Vendor_XXXX_2020/submission. 

**Movie S1. Large shifts in bacterial community structure occurred after clindamycin and *C. difficile* infection.** PCoA of $\theta_{YC}$ distances animated from 0 through 9 days post-infection. PERMANOVA analysis indicated source was the variable that explained the most observed variation across fecal communities (source R^2^ = `r rsq_adonis_source`, *P* = `r p_adonis_source`) followed by interactions between cage and day of the experiment. Transparency of the circle corresponds to the day of the experiment, each circle represents a sample from an individual mouse at a specific timepoint. See Table S2 for PERMANOVA results). Circles represent mice from experiment 1 and triangles represent mice from expeirment 2.

**Table S1. Diversity metrics Kruskal-Wallis and pairwise Wilcoxon statistical results.**

**Table S2. PERMANOVA results for mice at specific timepoints and across all timepoints.**

**Table S3. OTUs with relative abudances that significantly vary across sources at baseline, post-clindamycin, or post-infection timepoints.**

**Table S4. *C. difficile* CFU statistical results.**

**Table S5. Mouse weight change statistical results.**

**Table S6. OTUs with relative abudances that significantly changed after clindamycin treatment.**

**Table S7. Statistical results of L2-regularized logistic regression model performances.**

**Table S8. Top 20 most important OTUs for each of the 3 L2-regularized logistic regression models based on OTU relative abundance data.**

**Table S9. OTUs with relative abudances that significantly varied across sources of mice on at least 1 day of the experiment by Kruskal-Wallis test.**



